diff --git a/linux/localOptions.cfg b/linux/localOptions.cfg
index 311dca0..5c413d3 100644
--- a/linux/localOptions.cfg
+++ b/linux/localOptions.cfg
@@ -3,7 +3,15 @@
 [CentralServer]
 chatServiceBindInterface=eth0
 customerServiceBindInterface=eth0
 clusterName=CLUSTERNAME
+transferServerAddress=HOSTIP
+transferServerPort=44469
+
+[TransferServer]
+centralServerServiceBindInterface=eth0
+centralServerServiceBindPort=44469
+
+[CentralServer]
 
 webUpdateIntervalSeconds=600
 
@@ -51,6 +57,11 @@ startPlanet=space_npe_falcon
 startPlanet=space_npe_falcon_2
 startPlanet=space_npe_falcon_3
 startPlanet=space_ord_mantell
+startPlanet=space_ord_mantell_2
+startPlanet=space_ord_mantell_3
+startPlanet=space_ord_mantell_4
+startPlanet=space_ord_mantell_5
+startPlanet=space_ord_mantell_6
 
 [ServerMetrics]
 metricsServerPort=0
@@ -329,6 +335,6 @@ heroicTokenBonus=1
 
 ##BONUS: XP
-xpMultiplier=3
+xpMultiplier=1
 
 triggerVolumeSystem=2
 numberOfMoveObjectLists=10
