diff --git a/linux/localOptions.cfg b/linux/localOptions.cfg
index 311dca0..5c413d3 100644
--- a/linux/localOptions.cfg
+++ b/linux/localOptions.cfg
@@ -3,7 +3,13 @@
 [CentralServer]
 chatServiceBindInterface=eth0
 customerServiceBindInterface=eth0
 clusterName=CLUSTERNAME
+transferServerAddress=HOSTIP
+transferServerPort=44469
+
+[TransferServer]
+centralServerServiceBindInterface=eth0
+centralServerServiceBindPort=44469
 
 webUpdateIntervalSeconds=600
 
@@ -329,6 +335,6 @@ heroicTokenBonus=1
 
 ##BONUS: XP
-xpMultiplier=3
+xpMultiplier=1
 
 triggerVolumeSystem=2
 numberOfMoveObjectLists=10
