diff --git a/linux/localOptions.cfg b/linux/localOptions.cfg
index ec3c12a..484980a 100644
--- a/linux/localOptions.cfg
+++ b/linux/localOptions.cfg
@@ -133,6 +133,7 @@ MuseumEventDuration=1209600
 [SharedLog]
 #logTarget=net:HOSTIP:44467
 logTarget=file:logs/precuCombatCadence.log{c-*:c+PreCuCombatCadence}
+logTarget=file:logs/precuScoutHarvest.log{c-*:c+PreCuScoutHarvest}
 
 [SharedNetwork]
 oldestUnacknowledgedTimeout=0
