diff --git a/linux/localOptions.cfg b/linux/localOptions.cfg
--- a/linux/localOptions.cfg
+++ b/linux/localOptions.cfg
@@ -134,6 +134,7 @@ MuseumEventDuration=1209600
 #logTarget=net:HOSTIP:44467
 logTarget=file:logs/precuCombatCadence.log{c-*:c+PreCuCombatCadence}
 logTarget=file:logs/precuScoutHarvest.log{c-*:c+PreCuScoutHarvest}
+logTarget=file:logs/precuNpcConversation.log{c-*:c+PreCuConversation}
 
 [SharedNetwork]
 oldestUnacknowledgedTimeout=0
