diff --git a/linux/localOptions.cfg b/linux/localOptions.cfg
--- a/linux/localOptions.cfg
+++ b/linux/localOptions.cfg
@@ -135,6 +135,7 @@ MuseumEventDuration=1209600
 logTarget=file:logs/precuCombatCadence.log{c-*:c+PreCuCombatCadence}
 logTarget=file:logs/precuScoutHarvest.log{c-*:c+PreCuScoutHarvest}
 logTarget=file:logs/precuNpcConversation.log{c-*:c+PreCuConversation}
+logTarget=file:logs/precuObjectMenu.log{c-*:c+PreCuObjectMenu}
 
 [SharedNetwork]
 oldestUnacknowledgedTimeout=0
