diff --git a/sku.0/sys.server/compiled/game/script/theme_park/newbie_tutorial/newbie.java b/sku.0/sys.server/compiled/game/script/theme_park/newbie_tutorial/newbie.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/newbie_tutorial/newbie.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/newbie_tutorial/newbie.java
@@ -11,7 +11,6 @@ public class newbie extends script.theme_park.newbie_tutorial.tutorial_base
     public newbie()
     {
     }
-    public static final String questNewbieHall01 = "c_newbie_hall_01";
     public int OnAttach(obj_id self) throws InterruptedException
     {
         if (features.isSpaceEdition(self))
@@ -59,11 +58,6 @@ public class newbie extends script.theme_park.newbie_tutorial.tutorial_base
     }
     public void beginTutorial(obj_id self) throws InterruptedException
     {
-        obj_id bldg = getTopMostContainer(self);
-        if (!groundquests.isQuestActiveOrComplete(self, questNewbieHall01))
-        {
-            groundquests.grantQuest(self, questNewbieHall01, false);
-        }
         messageTo(self, "handleWelcome", null, 5, false);
     }
     public int OnDetach(obj_id self) throws InterruptedException
