diff --git a/sku.0/sys.server/compiled/game/datatables/spawning/poi_spawner/tatooine.tab b/sku.0/sys.server/compiled/game/datatables/spawning/poi_spawner/tatooine.tab
--- a/sku.0/sys.server/compiled/game/datatables/spawning/poi_spawner/tatooine.tab
+++ b/sku.0/sys.server/compiled/game/datatables/spawning/poi_spawner/tatooine.tab
@@ -347,6 +347,7 @@ s	c	c	c	c	c	f	i	s	f	s	i[1]	i[1]
 4936236	3438.84	5	-4915.72	tatooine	0		1	commoner_rodian_male		conversation.c_kashyyyk_hunter	1
 4936257	3435.53	4.105	-4845.76	tatooine	1082874		1	prof_quest_entertainer		conversation.c_prof_ent_questgiver	1
 4936258	3513.14	5	-4814.52	tatooine	0		1	prof_quest_artisan		conversation.c_prof_arti_questgiver	1
+91001001	3503	5	-4809	tatooine	0		1	trainer_artisan			1
 4936260	3436.44	7.5	-4719.83	tatooine	1279956		1	mos_eisley_mayor		conversation.mos_eisley_mayor	1
 2976495	3224.94	5.40827	-4852.27	tatooine	1188045		1	noble_male		conversation.c_prof_ent_host	1
 1095634	3840.2	180	-4498	tatooine	0	120	3	parry_stormtrooper_dewback	3		0
