diff --git a/sku.0/sys.shared/compiled/game/datatables/creation/attribute_limits.tab b/sku.0/sys.shared/compiled/game/datatables/creation/attribute_limits.tab
--- a/sku.0/sys.shared/compiled/game/datatables/creation/attribute_limits.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/creation/attribute_limits.tab
@@ -1,12 +1,12 @@
 male_template	female_template	min_health	max_health	min_strength	max_strength	min_constitution	max_constitution	min_action	max_action	min_quickness	max_quickness	min_stamina	max_stamina	min_mind	max_mind	min_focus	max_focus	min_willpower	max_willpower	total
 s	s	i	i	i	i	i	i	i	i	i	i	i	i	i	i	i	i	i	i	i
-bothan_male	bothan_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-human_male	human_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-moncal_male	moncal_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-rodian_male	rodian_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-trandoshan_male	trandoshan_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-twilek_male	twilek_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-wookiee_male	wookiee_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-zabrak_male	zabrak_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-sullustan_male	sullustan_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
-ithorian_male	ithorian_female	1000	1000	0	0	1000	1000	300	300	0	0	500	500	300	300	0	0	1000	1000	4100
+bothan_male	bothan_female	300	1000	300	500	300	400	600	1300	600	750	400	500	400	1100	400	600	300	500	5400
+human_male	human_female	400	1100	400	1100	400	1100	400	1100	400	1100	400	1100	400	1100	400	1100	400	1100	5400
+moncal_male	moncal_female	300	1000	300	500	300	400	300	1000	300	450	450	550	600	1300	600	800	450	650	5400
+rodian_male	rodian_female	300	1000	300	500	300	400	300	1200	300	650	450	850	300	1000	300	500	350	550	5400
+trandoshan_male	trandoshan_female	550	1250	600	800	700	800	300	1000	300	450	300	400	300	1000	300	500	300	600	5550
+twilek_male	twilek_female	300	1000	300	500	550	650	550	1250	600	750	300	400	400	1100	300	500	300	500	5400
+wookiee_male	wookiee_female	650	1350	650	850	450	550	500	1200	400	550	400	500	400	1100	450	650	400	600	6100
+zabrak_male	zabrak_female	500	1200	300	500	300	400	600	1300	300	450	300	400	300	1000	300	500	700	900	5400
+sullustan_male	sullustan_female	300	1200	300	500	300	400	600	1400	300	750	300	500	400	1200	400	600	300	600	5400
+ithorian_male	ithorian_female	300	1400	300	600	300	500	600	1100	300	750	300	500	400	1300	400	600	300	500	5400
diff --git a/sku.0/sys.shared/compiled/game/datatables/creation/profession_mods.tab b/sku.0/sys.shared/compiled/game/datatables/creation/profession_mods.tab
--- a/sku.0/sys.shared/compiled/game/datatables/creation/profession_mods.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/creation/profession_mods.tab
@@ -1,9 +1,9 @@
 profession	health	strength	constitution	action	quickness	stamina	mind	focus	willpower
 s	I	I	I	I	I	I	I	I	I
-crafting_artisan	1000	0	1000	300	0	500	300	0	1000
-combat_brawler	1000	0	1000	300	0	500	300	0	1000
-social_entertainer	1000	0	1000	300	0	500	300	0	1000
-combat_marksman	1000	0	1000	300	0	500	300	0	1000
-science_medic	1000	0	1000	300	0	500	300	0	1000
-outdoors_scout	1000	0	1000	300	0	500	300	0	1000
-jedi	1000	0	1000	300	0	500	300	0	1000
+crafting_artisan	600	300	300	800	400	300	900	400	500
+combat_brawler	1000	500	400	800	350	350	500	300	300
+social_entertainer	500	300	300	1000	400	400	800	400	400
+combat_marksman	1000	450	300	800	450	300	600	300	300
+science_medic	600	300	300	700	300	300	1000	500	500
+outdoors_scout	800	300	350	800	400	400	700	350	400
+jedi	1000	500	400	800	350	350	500	300	300
diff --git a/sku.0/sys.shared/compiled/game/datatables/creation/racial_mods.tab b/sku.0/sys.shared/compiled/game/datatables/creation/racial_mods.tab
--- a/sku.0/sys.shared/compiled/game/datatables/creation/racial_mods.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/creation/racial_mods.tab
@@ -1,12 +1,12 @@
 male_template	female_template	health	strength	constitution	action	quickness	stamina	mind	focus	willpower
 s	s	I	I	I	I	I	I	I	I	I
-bothan_male	bothan_female	0	0	0	0	0	0	0	0	0
-human_male	human_female	0	0	0	0	0	0	0	0	0
-moncal_male	moncal_female	0	0	0	0	0	0	0	0	0
-rodian_male	rodian_female	0	0	0	0	0	0	0	0	0
-trandoshan_male	trandoshan_female	0	0	0	0	0	0	0	0	0
-twilek_male	twilek_female	0	0	0	0	0	0	0	0	0
-wookiee_male	wookiee_female	0	0	0	0	0	0	0	0	0
-zabrak_male	zabrak_female	0	0	0	0	0	0	0	0	0
-sullustan_male	sullustan_female	0	0	0	0	0	0	0	0	0
-ithorian_male	ithorian_female	0	0	0	0	0	0	0	0	0
+bothan_male	bothan_female	0	0	0	300	300	100	100	100	0
+human_male	human_female	100	100	100	100	100	100	100	100	100
+moncal_male	moncal_female	0	0	0	0	0	150	300	300	150
+rodian_male	rodian_female	0	0	0	200	200	450	0	0	50
+trandoshan_male	trandoshan_female	250	300	400	0	0	0	0	0	100
+twilek_male	twilek_female	0	0	250	250	300	0	100	0	0
+wookiee_male	wookiee_female	350	350	150	200	100	100	100	150	100
+zabrak_male	zabrak_female	200	0	0	300	0	0	0	0	400
+sullustan_male	sullustan_female	150	0	0	250	50	50	200	100	100
+ithorian_male	ithorian_female	400	100	100	100	0	0	200	0	0
