diff --git a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
index f2620297d8896adf746ae4313e57b963814c70f9..839522f0bcc0ffa9f00bb67377c7866dc5e91bc2 100755
--- a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
+++ b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
@@ -14543,4 +14543,12 @@ public class combat_actions extends script.systems.combat.combat_base
         }
         return SCRIPT_CONTINUE;
     }
+    public int headShot1(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
+    {
+        if (!combatStandardAction("headShot1", self, target, params, "", ""))
+        {
+            return SCRIPT_OVERRIDE;
+        }
+        return SCRIPT_CONTINUE;
+    }
 }
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
index baa3891c642bfc93dbd6ff24c8a88fd5dc0cbab5..f17055f078a1e9d6cb54a047b6bd05f36154d327 100755
--- a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
@@ -1285,3 +1285,4 @@ of_del_ae_dm_boss	of_del_ae_dm_boss	BossOfficerOrbitalStrike				STANDARD	DELAY_A
 gcw_atst_attack_tower	gcw_atst_attack_tower				LEFT_CLICK_DEFAULT	STANDARD	ATTACK		1											0	0	*creature_attack_ranged														OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET	0	0			1000		4		1						0																																			NO_ATTACK_NAME
 gcw_rocket_launch	gcw_rocket_launch					STANDARD	DELAY_ATTACK			appearance/pt_gcw_rocket_launch.prt	root					0.5		TARGETED_OBJECT	STANDARD	1	1	fire_1_single_medium														OFF	OFF	OFF	OFF	ON	AREA	5			100	10000		1		1						1000					fire	1000	10																									gcw_rocket			
 firePulseCannon	firePulseCannon				LEFT_CLICK_DEFAULT	STANDARD	DELAY_ATTACK								0	0	1	LOCATION	STANDARD	1	0	fire_blue_energy_at_loc														OFF	OFF	OFF	OFF	OFF	TARGET_AREA	5			64											750																																			WEAPON_NAME
+headShot1	headShot1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_1_special_single								fire_1_special_single						OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	64	0	0	1.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				RIFLE	RANGED_WEAPON	ENERGY	KINETIC				marksman	0		ACTION_NAME	0	0	1
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
index b08fd04b4522f96ba0afd2964d49d917be13ffa0..f7e664a416b06db82d487c026a9f9bd78a5433f0 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
@@ -1,3 +1,4 @@
 actionName	healthCostMultiplier	actionCostMultiplier	mindCostMultiplier	targetPool
 s	f	f	f	"e(HEALTH=0,ACTION=1,MIND=2)[HEALTH]"
 __precu_runtime_probe	0	0	0	HEALTH
+headShot1	0.5	0.5	1	MIND
diff --git a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
index 6509dd6d4cf7e8a57b8ec3f506e710a65dafa765..bf910f614d50cb181a4275d336162269493c5288 100755
--- a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
@@ -1907,3 +1907,4 @@ callforsnowspeeder		normal	callForSnowspeederPickup					callforsnowspeeder	1	1	1
 callforslave1pickup		normal	callForTcgSlave1Pickup					callforslave1pickup	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0	0	1	0	0	1	1	1	1	1	1	1	1	1	1	1	0	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	other	optional		2	0	0	0	0	0	0	0	MELEE		defaultCooldownGroup	0	0	0		0	0	0
 costume_esb_yoda	commandClicky	normal	costume_esb_yoda	failSpecialAttack			1	costume_esb_yoda	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	1	0	0	0			0	0	1	1		1	0	0	1	1	1	1	0	1	0	1	1	1	1	0	0	1								0		0	0	0	0		1							other	none		2		costume	0			costume				costume	0	0.25	10				
 surrenderSkill		immediate			surrenderSkill		0		1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1		other	none		1	0	0	0	0	0	0	0	ALL	NONE	defaultCooldownGroup	0	0	0		0	0	0
+headShot1	combat	normal	headShot1	failSpecialAttack			1.5	headShot1	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	0	0	0	1	ALL	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
diff --git a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
index f15ef5f649958ade2d677616efcd20fa0b7914ad..39c9d72586f287ea5a0e371a944d0d3b625032b1 100755
--- a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
@@ -788,7 +788,7 @@ combat_brawler_polearm_04	combat_brawler_polearm_03		1	1	0	1	4000	5		combat_braw
 combat_marksman	combat		1	0	1	1	0	0						0											0	0
 combat_marksman_novice	combat_marksman		1	1	0	1	100	15					0	0							"private_marksman_novice,knockdownRecovery"	"ranged_accuracy=10,ranged_speed=5"			0	0
 combat_marksman_master	combat_marksman		1	1	0	1	5000	6		"combat_marksman_rifle_04,combat_marksman_pistol_04,combat_marksman_carbine_04,combat_marksman_support_04"		combat_general	35000	70000							"private_marksman_master,lethalShot"	"ranged_defense=5,ranged_accuracy=25,ranged_speed=15"			0	0
-combat_marksman_rifle_01	combat_marksman		1	0	0	1	1000	2		combat_marksman_novice		combat_rangedspecialize_rifle	1000	200000							private_marksman_rifle_1	"rifle_defense=5,rifle_accuracy=10,rifle_speed=5"			0	0
+combat_marksman_rifle_01	combat_marksman		1	0	0	1	1000	2		combat_marksman_novice		combat_rangedspecialize_rifle	1000	200000							"private_marksman_rifle_1,headShot1"	"rifle_defense=5,rifle_accuracy=10,rifle_speed=5"			0	0
 combat_marksman_rifle_02	combat_marksman_rifle_01		1	0	0	1	2000	3		combat_marksman_rifle_01		combat_rangedspecialize_rifle	5000	200000							"private_marksman_rifle_2,placedShot"	"rifle_defense=5,rifle_accuracy=10,rifle_speed=5"			0	0
 combat_marksman_rifle_03	combat_marksman_rifle_02		1	0	0	1	3000	4		combat_marksman_rifle_02		combat_rangedspecialize_rifle	15000	200000							"private_marksman_rifle_3,firearmStrike"	"rifle_defense=5,rifle_accuracy=10,rifle_speed=5"			0	0
 combat_marksman_rifle_04	combat_marksman_rifle_03		1	1	0	1	4000	5		combat_marksman_rifle_03		combat_rangedspecialize_rifle	70000	200000							"private_marksman_rifle_4,aim"	"rifle_defense=5,rifle_accuracy=10,rifle_speed=5"			0	0
