diff --git a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
index 702bd2701..fef44225d 100755
--- a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
@@ -387,7 +387,7 @@ social_dancer_shock_01	social_dancer	fourByFour	0	0	0	0	6000	5	0	social_dancer_n
 social_dancer_shock_02	social_dancer_shock_01	fourByFour	0	0	0	0	7000	4	0	social_dancer_shock_01		entertainer_healing	50000	400000		0			none			healing_dance_shock=10			1	0
 social_dancer_shock_03	social_dancer_shock_02	fourByFour	0	0	0	0	8000	3	0	social_dancer_shock_02		entertainer_healing	100000	500000		0			none			healing_dance_shock=20			1	0
 social_dancer_shock_04	social_dancer_shock_03	fourByFour	0	1	0	0	9000	2	0	social_dancer_shock_03		entertainer_healing	125000	500000		0			none			healing_dance_shock=25			1	0
-social_musician	social		0	0	1	0	0	0						0											0	0
+social_musician	social	fourByFour	0	0	1	0	0	0	0				0	0		0			none						1	0
 social_musician_novice	social_musician	fourByFour	0	1	0	0	5000	6	0	"social_entertainer_music_04,social_entertainer_healing_04"		music	50000	350000		0			none		"traz,registerWithLocation,startMusic+starwars4"	"healing_music_ability=5,healing_music_wound=5,healing_music_shock=10,healing_music_mind=10,instrument_assembly=10"	craftInstrumentGroupC		1	0
 social_musician_master	social_musician	fourByFour	0	1	0	0	10000	1	0	"social_musician_ability_04,social_musician_wound_04,social_musician_knowledge_04,social_musician_shock_04"			0	0		0			none		"startMusic+virtuoso,nalargon,place_cantina,place_theater"	"healing_music_ability=15,healing_music_wound=15,healing_music_shock=25,healing_music_mind=25,instrument_assembly=25,ranged_defense=7,melee_defense=7,private_place_cantina=100,private_place_theater=100"			1	0
 social_musician_ability_01	social_musician	fourByFour	0	0	0	0	6000	5	0	social_musician_novice		music	87500	500000		0			none		"spotlight,colorlights,dazzle"	"instrument_assembly=10,healing_music_mind=10"			1	0
