diff --git a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
index d01fb0925..80edf2af1 100755
--- a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
@@ -747,7 +747,7 @@ outdoors_scout_camp_03	outdoors_scout_camp_02		1	0	0	1	3000	4		outdoors_scout_ca
 outdoors_scout_camp_04	outdoors_scout_camp_03		1	1	0	1	4000	5		outdoors_scout_camp_03		scout	25000	75000							private_scout_camp_4	camp=20	craftScoutCampGroupE		0	0
 science			1	0	0	1	0	0						0											0	0
 science_medic	science	fourByFour	0	0	1	0	0	0	0				0	0		0			none						1	0
-science_medic_novice	science_medic		0	1	0	0	100	15	0				0	0		0					"private_medic_novice,healDamage,healWound,medicalForage,tendWound,tendDamage,diagnose"	"healing_injury_treatment=5,healing_wound_treatment=5,healing_ability=5,healing_injury_speed=5,medicine_assembly=10,medicine_experimentation=10,medical_foraging=10"	"craftMedicineComponentGroupA,craftMedicineStimpackGroupA"		1	0
+science_medic_novice	science_medic	fourByFour	0	1	0	0	100	15	0				0	0		0			none		"private_medic_novice,healDamage,healWound,medicalForage,tendWound,tendDamage,diagnose"	"healing_injury_treatment=5,healing_wound_treatment=5,healing_ability=5,healing_injury_speed=5,medicine_assembly=10,medicine_experimentation=10,medical_foraging=10"	"craftMedicineComponentGroupA,craftMedicineStimpackGroupA"		1	0
 science_medic_master	science_medic	fourByFour	0	1	0	0	5000	6	0	"science_medic_injury_04,science_medic_injury_speed_04,science_medic_ability_04,science_medic_crafting_04"			0	0		0			none		private_medic_master	"healing_injury_treatment=25,healing_ability=10,medical_foraging=25"	craftMedicineStimpackGroupD		1	0
 science_medic_injury_01	science_medic	fourByFour	0	0	0	0	1000	2	0	science_medic_novice		medical	1000	10000		0			none		private_medic_injury_1	healing_injury_treatment=15			1	0
 science_medic_injury_02	science_medic_injury_01	fourByFour	0	0	0	0	2000	3	0	science_medic_injury_01		medical	5000	20000		0			none		"private_medic_injury_2,firstAid"	healing_injury_treatment=15			1	0
