diff --git a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
index e5101c3cd..069fcb8d2 100755
--- a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
@@ -995,23 +995,23 @@ science_doctor_prereq_04	science_doctor_prereq_03	oneByFour	1	0	0	1	4000	5		scie
 science_doctor_prereq_master	science_doctor_prereq	oneByFour	1	0	0	1	5000	6		science_doctor_prereq_04		combat_general	85000	150000							"bactaSpray_2,neuroToxin_2"	"buffing_efficiency=2,healing_efficiency=2"			0	0
 science_doctor	science	fourByFour	0	0	1	0	0	0	0				0	0		0			none						1	0
 science_doctor_novice	science_doctor	fourByFour	0	1	0	0	5000	6	0	science_medic_master		medical	11250	20000		0			none		"healState,registerWithLocation"	"healing_wound_treatment=5,healing_wound_speed=5,medicine_assembly=10,medicine_experimentation=10,healing_ability=5"	"craftCurePoisonGroupA,craftStimpackStateGroupA"		1	0
-science_doctor_master	science_doctor		0	1	0	0	10000	1	0	"science_doctor_wound_speed_04,science_doctor_wound_04,science_doctor_ability_04,science_doctor_support_04"			0	0		0					place_hospital	"healing_wound_treatment=25,healing_wound_speed=25,healing_ability=10,private_place_hospital=100"	"craftMedpackGroupE,craftMedpackBGroupE,craftCureDiseaseGroupC,craftMedpackBEnhanceGroupD"		1	0
-science_doctor_wound_speed_01	science_doctor		0	0	0	0	6000	5	0	science_doctor_novice		medical	15000	40000		0						healing_wound_speed=15			1	0
-science_doctor_wound_speed_02	science_doctor_wound_speed_01		0	0	0	0	7000	4	0	science_doctor_wound_speed_01		medical	30000	60000		0					curePoison	healing_wound_speed=15			1	0
-science_doctor_wound_speed_03	science_doctor_wound_speed_02		0	0	0	0	8000	3	0	science_doctor_wound_speed_02		medical	45000	80000		0					extinguishFire	healing_wound_speed=15			1	0
-science_doctor_wound_speed_04	science_doctor_wound_speed_03		0	1	0	0	9000	2	0	science_doctor_wound_speed_03		medical	60000	80000		0					cureDisease	healing_wound_speed=25			1	0
-science_doctor_wound_01	science_doctor		0	0	0	0	6000	5	0	science_doctor_novice		medical	15000	40000		0						healing_wound_treatment=10			1	0
-science_doctor_wound_02	science_doctor_wound_01		0	0	0	0	7000	4	0	science_doctor_wound_01		medical	30000	60000		0					healEnhance	healing_wound_treatment=15			1	0
-science_doctor_wound_03	science_doctor_wound_02		0	0	0	0	8000	3	0	science_doctor_wound_02		medical	45000	80000		0						healing_wound_treatment=15			1	0
-science_doctor_wound_04	science_doctor_wound_03		0	1	0	0	9000	2	0	science_doctor_wound_03		medical	60000	80000		0					revivePlayer	healing_wound_treatment=25			1	0
-science_doctor_ability_01	science_doctor		0	0	0	0	6000	5	0	science_doctor_novice		medical	15000	40000		0						healing_ability=5			1	0
-science_doctor_ability_02	science_doctor_ability_01		0	0	0	0	7000	4	0	science_doctor_ability_01		medical	30000	60000		0						healing_ability=10			1	0
-science_doctor_ability_03	science_doctor_ability_02		0	0	0	0	8000	3	0	science_doctor_ability_02		medical	45000	80000		0						healing_ability=10			1	0
-science_doctor_ability_04	science_doctor_ability_03		0	1	0	0	9000	2	0	science_doctor_ability_03		medical	60000	80000		0						healing_ability=10			1	0
-science_doctor_support_01	science_doctor		0	0	0	0	6000	5	0	science_doctor_novice		crafting_medicine_general	15000	28000		0						"medicine_assembly=10,medicine_experimentation=10"	"craftCureDiseaseGroupA,craftMedpackEnhanceGroupA,craftStimpackStateGroupB"		1	0
-science_doctor_support_02	science_doctor_support_01		0	0	0	0	7000	4	0	science_doctor_support_01		crafting_medicine_general	21000	36000		0						"medicine_assembly=10,medicine_experimentation=10"	"craftMedpackGroupC,craftCurePoisonGroupB,craftMedpackEnhanceGroupB,craftMedpackBEnhanceGroupA"		1	0
-science_doctor_support_03	science_doctor_support_02		0	0	0	0	8000	3	0	science_doctor_support_02		crafting_medicine_general	27000	44000		0						"medicine_assembly=10,medicine_experimentation=10"	"craftMedicineStimpackGroupE,craftMedpackBGroupC,craftCureDiseaseGroupB,craftMedpackEnhanceGroupC,craftMedpackBEnhanceGroupB"		1	0
-science_doctor_support_04	science_doctor_support_03		0	1	0	0	9000	2	0	science_doctor_support_03		crafting_medicine_general	33000	44000		0						"medicine_assembly=10,medicine_experimentation=10"	"craftMedpackGroupD,craftMedpackBGroupD,craftCurePoisonGroupC,craftMedpackEnhanceGroupD,craftMedpackBEnhanceGroupC"		1	0
+science_doctor_master	science_doctor	fourByFour	0	1	0	0	10000	1	0	"science_doctor_wound_speed_04,science_doctor_wound_04,science_doctor_ability_04,science_doctor_support_04"			0	0		0			none		place_hospital	"healing_wound_treatment=25,healing_wound_speed=25,healing_ability=10,private_place_hospital=100"	"craftMedpackGroupE,craftMedpackBGroupE,craftCureDiseaseGroupC,craftMedpackBEnhanceGroupD"		1	0
+science_doctor_wound_speed_01	science_doctor	fourByFour	0	0	0	0	6000	5	0	science_doctor_novice		medical	15000	40000		0			none			healing_wound_speed=15			1	0
+science_doctor_wound_speed_02	science_doctor_wound_speed_01	fourByFour	0	0	0	0	7000	4	0	science_doctor_wound_speed_01		medical	30000	60000		0			none		curePoison	healing_wound_speed=15			1	0
+science_doctor_wound_speed_03	science_doctor_wound_speed_02	fourByFour	0	0	0	0	8000	3	0	science_doctor_wound_speed_02		medical	45000	80000		0			none		extinguishFire	healing_wound_speed=15			1	0
+science_doctor_wound_speed_04	science_doctor_wound_speed_03	fourByFour	0	1	0	0	9000	2	0	science_doctor_wound_speed_03		medical	60000	80000		0			none		cureDisease	healing_wound_speed=25			1	0
+science_doctor_wound_01	science_doctor	fourByFour	0	0	0	0	6000	5	0	science_doctor_novice		medical	15000	40000		0			none			healing_wound_treatment=10			1	0
+science_doctor_wound_02	science_doctor_wound_01	fourByFour	0	0	0	0	7000	4	0	science_doctor_wound_01		medical	30000	60000		0			none		healEnhance	healing_wound_treatment=15			1	0
+science_doctor_wound_03	science_doctor_wound_02	fourByFour	0	0	0	0	8000	3	0	science_doctor_wound_02		medical	45000	80000		0			none			healing_wound_treatment=15			1	0
+science_doctor_wound_04	science_doctor_wound_03	fourByFour	0	1	0	0	9000	2	0	science_doctor_wound_03		medical	60000	80000		0			none		revivePlayer	healing_wound_treatment=25			1	0
+science_doctor_ability_01	science_doctor	fourByFour	0	0	0	0	6000	5	0	science_doctor_novice		medical	15000	40000		0			none			healing_ability=5			1	0
+science_doctor_ability_02	science_doctor_ability_01	fourByFour	0	0	0	0	7000	4	0	science_doctor_ability_01		medical	30000	60000		0			none			healing_ability=10			1	0
+science_doctor_ability_03	science_doctor_ability_02	fourByFour	0	0	0	0	8000	3	0	science_doctor_ability_02		medical	45000	80000		0			none			healing_ability=10			1	0
+science_doctor_ability_04	science_doctor_ability_03	fourByFour	0	1	0	0	9000	2	0	science_doctor_ability_03		medical	60000	80000		0			none			healing_ability=10			1	0
+science_doctor_support_01	science_doctor	fourByFour	0	0	0	0	6000	5	0	science_doctor_novice		crafting_medicine_general	15000	28000		0			none			"medicine_assembly=10,medicine_experimentation=10"	"craftCureDiseaseGroupA,craftMedpackEnhanceGroupA,craftStimpackStateGroupB"		1	0
+science_doctor_support_02	science_doctor_support_01	fourByFour	0	0	0	0	7000	4	0	science_doctor_support_01		crafting_medicine_general	21000	36000		0			none			"medicine_assembly=10,medicine_experimentation=10"	"craftMedpackGroupC,craftCurePoisonGroupB,craftMedpackEnhanceGroupB,craftMedpackBEnhanceGroupA"		1	0
+science_doctor_support_03	science_doctor_support_02	fourByFour	0	0	0	0	8000	3	0	science_doctor_support_02		crafting_medicine_general	27000	44000		0			none			"medicine_assembly=10,medicine_experimentation=10"	"craftMedicineStimpackGroupE,craftMedpackBGroupC,craftCureDiseaseGroupB,craftMedpackEnhanceGroupC,craftMedpackBEnhanceGroupB"		1	0
+science_doctor_support_04	science_doctor_support_03	fourByFour	0	1	0	0	9000	2	0	science_doctor_support_03		crafting_medicine_general	33000	44000		0			none			"medicine_assembly=10,medicine_experimentation=10"	"craftMedpackGroupD,craftMedpackBGroupD,craftCurePoisonGroupC,craftMedpackEnhanceGroupD,craftMedpackBEnhanceGroupC"		1	0
 outdoors_ranger_prereq	outdoors	oneByFour	1	0	1	1	0	0						0											0	0
 outdoors_ranger_prereq_novice	outdoors_ranger_prereq	oneByFour	1	1	0	1	100	10				combat_general	1000	10000							terminateTarget	"ranged_accuracy=10,ranged_speed=5"			0	0
 outdoors_ranger_prereq_01	outdoors_ranger_prereq	oneByFour	1	0	0	1	1000	2		outdoors_ranger_prereq_novice		combat_general	6000	20000							smokeGrenade	"ranged_defense=5,ranged_accuracy=5,slope_move=5"			0	0
