diff --git a/sku.0/sys.server/compiled/game/script/player/base/base_player.java b/sku.0/sys.server/compiled/game/script/player/base/base_player.java
--- a/sku.0/sys.server/compiled/game/script/player/base/base_player.java
+++ b/sku.0/sys.server/compiled/game/script/player/base/base_player.java
@@ -474,13 +474,7 @@ public class base_player extends script.base_script
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        if ((utils.isProfession(self, utils.TRADER)) && (!hasSkill(self, "class_trader")))
-        {
-            grantSkill(self, "class_trader");
-        }
         city.initCitizen(self);
-        respecNewEntertainerSkills(self);
-        respecNewCrafterSkills(self);
         boolean setting = utils.checkConfigFlag("TestCenterPlayer", "ApplyTCMark");
         if (setting)
         {
@@ -492,11 +486,6 @@ public class base_player extends script.base_script
                 setJediState(self, JEDI_STATE_FORCE_SENSITIVE);
             }
         }
-        if (hasSkill(self, "jedi_padawan_novice"))
-        {
-            setSkillTemplate(self, "a");
-            attachScript(self, "player.player_jedi_conversion");
-        }
         if (!isJedi(self))
         {
             if (hasObjVar(self, "jedi.postponeGrant"))
