diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/loot.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/loot.java
index 0a1ac30..14711f6 100644
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/loot.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/loot.java
@@ -7,7 +7,6 @@ import script.library.weapons;
 import script.location;
 import script.menu_info_types;
 import script.obj_id;
-import script.string_id;
 
 public class loot extends script.base_script
 {
@@ -43,12 +42,6 @@ public class loot extends script.base_script
     }
     public int OnObjectMenuSelect(obj_id self, obj_id player, int item) throws InterruptedException
     {
-        String classTemplate = getSkillTemplate(player);
-        if (classTemplate == null || classTemplate.startsWith("entertainer") || classTemplate.startsWith("trader"))
-        {
-            sendSystemMessage(player, new string_id("dungeon/corvette", "no_trader_farming_allowed"));
-            return SCRIPT_CONTINUE;
-        }
         if (item == menu_info_types.ITEM_OPEN)
         {
             if (!hasObjVar(self, "used"))
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/disk_loot.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/disk_loot.java
index 0bef53b..590f209 100644
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/disk_loot.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/disk_loot.java
@@ -5,7 +5,6 @@ import script.library.permissions;
 import script.library.utils;
 import script.menu_info_types;
 import script.obj_id;
-import script.string_id;
 
 public class disk_loot extends script.base_script
 {
@@ -27,12 +26,6 @@ public class disk_loot extends script.base_script
     }
     public int OnObjectMenuSelect(obj_id self, obj_id player, int item) throws InterruptedException
     {
-        String classTemplate = getSkillTemplate(player);
-        if (classTemplate == null || classTemplate.startsWith("entertainer") || classTemplate.startsWith("trader"))
-        {
-            sendSystemMessage(player, new string_id("dungeon/corvette", "no_trader_farming_allowed"));
-            return SCRIPT_CONTINUE;
-        }
         if (item == menu_info_types.ITEM_OPEN)
         {
             setOwner(self, player);
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/r2_loot.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/r2_loot.java
index 7572520..88e75be 100644
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/r2_loot.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/corvette/r2_loot.java
@@ -5,7 +5,6 @@ import script.library.permissions;
 import script.library.utils;
 import script.menu_info_types;
 import script.obj_id;
-import script.string_id;
 
 public class r2_loot extends script.base_script
 {
@@ -27,12 +26,6 @@ public class r2_loot extends script.base_script
     }
     public int OnObjectMenuSelect(obj_id self, obj_id player, int item) throws InterruptedException
     {
-        String classTemplate = getSkillTemplate(player);
-        if (classTemplate == null || classTemplate.startsWith("entertainer") || classTemplate.startsWith("trader"))
-        {
-            sendSystemMessage(player, new string_id("dungeon/corvette", "no_trader_farming_allowed"));
-            return SCRIPT_CONTINUE;
-        }
         if (item == menu_info_types.ITEM_OPEN)
         {
             if (!hasObjVar(self, permissions.VAR_PERMISSION_BASE))
