diff --git a/sku.0/sys.server/compiled/game/script/systems/crafting/item/crafting_gcw_tools.java b/sku.0/sys.server/compiled/game/script/systems/crafting/item/crafting_gcw_tools.java
index de7e7f2..48899db 100644
--- a/sku.0/sys.server/compiled/game/script/systems/crafting/item/crafting_gcw_tools.java
+++ b/sku.0/sys.server/compiled/game/script/systems/crafting/item/crafting_gcw_tools.java
@@ -1,6 +1,5 @@
 package script.systems.crafting.item;
 
-import script.library.skill;
 import script.library.utils;
 import script.obj_id;
 
@@ -19,15 +18,6 @@ public class crafting_gcw_tools extends script.base_script
         int count = getCount(self);
         int power = (int)getFloatObjVar(self, "crafting_components.power");
         int fatigue = 1;
-        String skillTemplate = getSkillTemplate(player);
-        String profession = skill.getProfessionName(skillTemplate);
-        if(profession == null){
-            LOG("gcw-crafting","Unable to get player's (" + player + ":" + getPlayerFullName(player) + ") profession!  Skill template is " + skillTemplate + ".");
-        }
-        else if (!profession.equals("trader"))
-        {
-            fatigue = 5;
-        }
         if (getCount(self) > 1)
         {
             names[idx] = "charges";
