diff --git a/sku.0/sys.server/compiled/game/script/item/wearable/senator_crate.java b/sku.0/sys.server/compiled/game/script/item/wearable/senator_crate.java
--- a/sku.0/sys.server/compiled/game/script/item/wearable/senator_crate.java
+++ b/sku.0/sys.server/compiled/game/script/item/wearable/senator_crate.java
@@ -35,7 +35,6 @@ public class senator_crate extends script.base_script
     {
         obj_id pInv = utils.getInventoryContainer(player);
         int pSpecies = getSpecies(player);
-        String playerTemplate = getSkillTemplate(player);
         int pGender = getGender(player);
         HashSet theSet = new HashSet();
         if (pSpecies == SPECIES_WOOKIEE)
diff --git a/sku.0/sys.server/compiled/game/script/item/ice_cream_fryer/fryer.java b/sku.0/sys.server/compiled/game/script/item/ice_cream_fryer/fryer.java
--- a/sku.0/sys.server/compiled/game/script/item/ice_cream_fryer/fryer.java
+++ b/sku.0/sys.server/compiled/game/script/item/ice_cream_fryer/fryer.java
@@ -293,20 +293,6 @@ public class fryer extends script.base_script
                     boolean success = setFourthIngredient(fryer, generatedItem, row);
                     return generatedItem;
                 }
-                else 
-                {
-                    blog("setBuffTypeOnEdible - The player getSkillTemplate: " + getSkillTemplate(player));
-                    if ((getSkillTemplate(player)).startsWith("trader_0a"))
-                    {
-                        CustomerServiceLog("ice_cream_fryer", "%TU has created a TRADER buff consumable using fryer: " + fryer, player);
-                        blog("setBuffTypeOnEdible - DOMESTICS BUFF BEING CREATED: " + i);
-                        generatedItem = static_item.createNewItemFunction(currentBuff, player);
-                        setObjVar(generatedItem, collection.FRYER_BUFF_TYPE, collection.ICE_CREAM_DOMESTICS_ONLY);
-                        boolean success = setFourthIngredient(fryer, generatedItem, row);
-                        return generatedItem;
-                    }
-                    blog("setBuffTypeOnEdible - DOMESTICS BUFF FAILED: ");
-                }
             }
         }
         CustomerServiceLog("ice_cream_fryer", "%TU has created a DEBUFF consumable using fryer: " + fryer, player);
