diff --git a/sku.0/sys.server/compiled/game/script/library/gcw.java b/sku.0/sys.server/compiled/game/script/library/gcw.java
--- a/sku.0/sys.server/compiled/game/script/library/gcw.java
+++ b/sku.0/sys.server/compiled/game/script/library/gcw.java
@@ -2558,8 +2558,8 @@ public class gcw extends script.base_script
             }
         }
         playerFaction = factionColor + playerFaction;
-        String playerProfession = "@ui_roadmap:" + skill.getProfessionName(getSkillTemplate(who));
-        int playerLevel = getLevel(who);
+        String playerProfession = "Publish 14.1 skills";
+        int playerLevel = 0;
         if (currentData != null)
         {
             playerGCW = currentData.playerGCW + playerGCW;
diff --git a/sku.0/sys.server/compiled/game/script/player/player_instance.java b/sku.0/sys.server/compiled/game/script/player/player_instance.java
--- a/sku.0/sys.server/compiled/game/script/player/player_instance.java
+++ b/sku.0/sys.server/compiled/game/script/player/player_instance.java
@@ -339,9 +339,7 @@ public class player_instance extends script.base_script
         int calendarTime = getCalendarTime();
         String realTime = getCalendarTimeStringLocal(calendarTime);
         int duration = data.getInt("time_limit");
-        String strProfession = skill.getProfessionName(getSkillTemplate(self));
         String playerName = getPlayerName(self);
-        int playerLevel = getLevel(self);
         obj_id groupId = getGroupObject(self);
         String playerFaction = factions.getFaction(self);
         int remaining = (start_time + duration) - getGameTime();
@@ -352,7 +350,7 @@ public class player_instance extends script.base_script
             CustomerServiceLog("instance-" + instance_name, "Start Time: " + realTime + ",  Duration: " + duration);
             CustomerServiceLog("instance-" + instance_name, "Instance Physical Location: " + instanceLoc);
             CustomerServiceLog("instance-" + instance_name, "Side " + team);
-            CustomerServiceLog("instance-" + instance_name, "Players Is: " + strProfession + "/" + playerLevel + "/" + playerFaction);
+            CustomerServiceLog("instance-" + instance_name, "Progression: Publish 14.1 skills, Faction: " + playerFaction);
             if (isIdValid(groupId))
             {
                 obj_id[] players = getGroupMemberIds(groupId);
@@ -372,7 +370,7 @@ public class player_instance extends script.base_script
             CustomerServiceLog("instance-" + instance_name, "Player Joined at: " + realTime + ", time remaining: " + remaining);
             CustomerServiceLog("instance-" + instance_name, "Instance Physical Location: " + instanceLoc);
             CustomerServiceLog("instance-" + instance_name, "Side " + team);
-            CustomerServiceLog("instance-" + instance_name, "Players Is: " + strProfession + "/" + playerLevel + "/" + playerFaction);
+            CustomerServiceLog("instance-" + instance_name, "Progression: Publish 14.1 skills, Faction: " + playerFaction);
             if (isIdValid(groupId))
             {
                 obj_id[] players = getGroupMemberIds(groupId);
