diff --git a/sku.0/sys.server/compiled/game/script/theme_park/heroic/tusken/tusken_controller.java b/sku.0/sys.server/compiled/game/script/theme_park/heroic/tusken/tusken_controller.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/heroic/tusken/tusken_controller.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/heroic/tusken/tusken_controller.java
@@ -25,8 +25,7 @@ public class tusken_controller extends script.base_script
         CustomerServiceLog("instance-heroic_tusken_army", "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog("instance-heroic_tusken_army", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog("instance-heroic_tusken_army", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         return SCRIPT_CONTINUE;
     }
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/heroic/axkva_min/axkva_controller.java b/sku.0/sys.server/compiled/game/script/theme_park/heroic/axkva_min/axkva_controller.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/heroic/axkva_min/axkva_controller.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/heroic/axkva_min/axkva_controller.java
@@ -26,8 +26,7 @@ public class axkva_controller extends script.base_script
         CustomerServiceLog("instance-heroic_axkva_min", "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog("instance-heroic_axkva_min", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog("instance-heroic_axkva_min", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         instance.setClock(self, 300);
         return SCRIPT_CONTINUE;
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/heroic/star_destroyer/sd_controller.java b/sku.0/sys.server/compiled/game/script/theme_park/heroic/star_destroyer/sd_controller.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/heroic/star_destroyer/sd_controller.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/heroic/star_destroyer/sd_controller.java
@@ -24,8 +24,7 @@ public class sd_controller extends script.base_script
         CustomerServiceLog("instance-heroic_star_destroyer", "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog("instance-heroic_star_destroyer", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog("instance-heroic_star_destroyer", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         return SCRIPT_CONTINUE;
     }
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/heroic/ig88/ig88.java b/sku.0/sys.server/compiled/game/script/theme_park/heroic/ig88/ig88.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/heroic/ig88/ig88.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/heroic/ig88/ig88.java
@@ -536,8 +536,7 @@ public class ig88 extends script.base_script
         CustomerServiceLog("instance-heroic_ig88", "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog("instance-heroic_ig88", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog("instance-heroic_ig88", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         instance.setClock(trial.getParent(self), 300);
         return SCRIPT_CONTINUE;
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/heroic/exar_kun/exar_controller.java b/sku.0/sys.server/compiled/game/script/theme_park/heroic/exar_kun/exar_controller.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/heroic/exar_kun/exar_controller.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/heroic/exar_kun/exar_controller.java
@@ -25,8 +25,7 @@ public class exar_controller extends script.base_script
         CustomerServiceLog("instance-heroic_exar_kun", "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog("instance-heroic_exar_kun", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog("instance-heroic_exar_kun", "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         instance.setClock(self, 300);
         return SCRIPT_CONTINUE;
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/heroic/echo_base/echo_controller.java b/sku.0/sys.server/compiled/game/script/theme_park/heroic/echo_base/echo_controller.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/heroic/echo_base/echo_controller.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/heroic/echo_base/echo_controller.java
@@ -144,8 +144,7 @@ public class echo_controller extends script.base_script
         CustomerServiceLog(instanceName, "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog(instanceName, "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog(instanceName, "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         CustomerServiceLog(instanceName, "Players earned " + tokens + " tokens");
         CustomerServiceLog(instanceName, "Objective Destroy 4 AT-AT: " + utils.getIntScriptVar(self, "quest_tracker.rebel.at_minor"));
@@ -182,8 +181,7 @@ public class echo_controller extends script.base_script
         CustomerServiceLog(instanceName, "Group (" + group + ") consists of: ");
         for (int i = 0; i < players.length; ++i)
         {
-            String strProfession = skill.getProfessionName(getSkillTemplate(players[i]));
-            CustomerServiceLog(instanceName, "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") profession is " + strProfession + ".");
+            CustomerServiceLog(instanceName, "Group (" + group + ") member " + i + " " + getFirstName(players[i]) + "'s(" + players[i] + ") progression is Publish 14.1 skills.");
         }
         CustomerServiceLog(instanceName, "Players earned " + tokens + " tokens");
         CustomerServiceLog(instanceName, "Objective Lost 1 AT-AT: " + utils.getIntScriptVar(self, "quest_tracker.imperial.fail_major"));
