diff --git a/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_control_device.java b/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_control_device.java
--- a/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_control_device.java
+++ b/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_control_device.java
@@ -14,21 +14,10 @@ public class quest_control_device extends script.base_script
     }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info menuInfo) throws InterruptedException
     {
-        if (utils.isNestedWithin(self, player))
-        {
-            int rootMenu = menuInfo.addRootMenu(menu_info_types.SERVER_MENU1, new string_id("saga_system", "qcd_menu_abandon"));
-        }
         return SCRIPT_CONTINUE;
     }
     public int OnObjectMenuSelect(obj_id self, obj_id player, int menu_item) throws InterruptedException
     {
-        if (menu_item == menu_info_types.SERVER_MENU1)
-        {
-            if (utils.isNestedWithin(self, player))
-            {
-                pgc_quests.setQuestAbandoned(self, player);
-            }
-        }
         return SCRIPT_CONTINUE;
     }
     public int OnGetAttributes(obj_id self, obj_id player, String[] names, String[] attribs) throws InterruptedException
diff --git a/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_holocron.java b/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_holocron.java
--- a/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_holocron.java
+++ b/sku.0/sys.server/compiled/game/script/quest/task/pgc/quest_holocron.java
@@ -15,9 +15,14 @@ public class quest_holocron extends script.base_script
     public static final int MENU_EDIT_RECIPE = menu_info_types.SERVER_MENU5;
     public static final int MENU_SHARE_QUEST = menu_info_types.SERVER_MENU6;
     public static final int MENU_BUILD_HOLOCRON = menu_info_types.SERVER_MENU7;
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "quest.task.pgc.quest_holocron");
+        return SCRIPT_CONTINUE;
+    }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        messageTo(self, "handleCheckCompletedHolocron", null, 2, false);
+        detachScript(self, "quest.task.pgc.quest_holocron");
         return SCRIPT_CONTINUE;
     }
     public int handleCheckCompletedHolocron(obj_id self, dictionary params) throws InterruptedException
@@ -29,6 +34,8 @@ public class quest_holocron extends script.base_script
     }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info menuInfo) throws InterruptedException
     {
+        return SCRIPT_CONTINUE;
+        /*
         int rootMenu = 0;
         obj_id playerInventory = utils.getInventoryContainer(player);
         if (utils.isNestedWithin(self, playerInventory))
@@ -82,10 +89,12 @@ public class quest_holocron extends script.base_script
                 rootMenu = menuInfo.addRootMenu(MENU_SHARE_QUEST, new string_id("saga_system", "holocron_menu_share_quest"));
             }
         }
-        return SCRIPT_CONTINUE;
+        */
     }
     public int OnObjectMenuSelect(obj_id self, obj_id player, int menu_item) throws InterruptedException
     {
+        return SCRIPT_CONTINUE;
+        /*
         obj_id playerInventory = utils.getInventoryContainer(player);
         if (utils.isNestedWithin(self, playerInventory))
         {
@@ -258,7 +267,7 @@ public class quest_holocron extends script.base_script
                 }
             }
         }
-        return SCRIPT_CONTINUE;
+        */
     }
     public boolean hasRewardItems(obj_id questHolocron) throws InterruptedException
     {
diff --git a/sku.0/sys.server/compiled/game/script/quest/task/pgc/credit_item.java b/sku.0/sys.server/compiled/game/script/quest/task/pgc/credit_item.java
--- a/sku.0/sys.server/compiled/game/script/quest/task/pgc/credit_item.java
+++ b/sku.0/sys.server/compiled/game/script/quest/task/pgc/credit_item.java
@@ -12,6 +12,7 @@ public class credit_item extends script.base_script
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
+        detachScript(self, "quest.task.pgc.credit_item");
         return SCRIPT_CONTINUE;
     }
     public int OnGetAttributes(obj_id self, obj_id player, String[] names, String[] attribs) throws InterruptedException
@@ -28,26 +29,10 @@ public class credit_item extends script.base_script
     }
     public int OnObjectMenuRequest(obj_id self, obj_id objPlayer, menu_info mi) throws InterruptedException
     {
-        mi.addRootMenu(menu_info_types.ITEM_USE, new string_id("space/space_loot", "use_credit_chip"));
         return SCRIPT_CONTINUE;
     }
     public int OnObjectMenuSelect(obj_id self, obj_id player, int item) throws InterruptedException
     {
-        obj_id containingPlayer = utils.getContainingPlayer(self);
-        if (!isIdValid(containingPlayer) || containingPlayer != player)
-        {
-            return SCRIPT_CONTINUE;
-        }
-        if (item == menu_info_types.ITEM_USE)
-        {
-            int cash = getIntObjVar(self, "loot.cashAmount");
-            money.bankTo("pgc_player_donated_credits", player, cash);
-            string_id creditsMsg = new string_id("saga_system", "holocron_reward_credits_given");
-            prose_package pp = prose.getPackage(creditsMsg, player, player);
-            prose.setDI(pp, cash);
-            sendSystemMessageProse(player, pp);
-            destroyObject(self);
-        }
         return SCRIPT_CONTINUE;
     }
 }
diff --git a/sku.0/sys.server/compiled/game/script/conversation/chronicles_reward_vendor.java b/sku.0/sys.server/compiled/game/script/conversation/chronicles_reward_vendor.java
--- a/sku.0/sys.server/compiled/game/script/conversation/chronicles_reward_vendor.java
+++ b/sku.0/sys.server/compiled/game/script/conversation/chronicles_reward_vendor.java
@@ -28,27 +28,19 @@ public class chronicles_reward_vendor extends script.base_script
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        if ((!isTangible(self)) || (isPlayer(self)))
-        {
-            detachScript(self, "conversation.chronicles_reward_vendor");
-        }
-        setCondition(self, CONDITION_CONVERSABLE);
-        ai_lib.setDefaultCalmBehavior(self, ai_lib.BEHAVIOR_SENTINEL);
+        clearCondition(self, CONDITION_CONVERSABLE);
+        detachScript(self, "conversation.chronicles_reward_vendor");
         return SCRIPT_CONTINUE;
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        setCondition(self, CONDITION_CONVERSABLE);
-        ai_lib.setDefaultCalmBehavior(self, ai_lib.BEHAVIOR_SENTINEL);
+        clearCondition(self, CONDITION_CONVERSABLE);
+        detachScript(self, "conversation.chronicles_reward_vendor");
         return SCRIPT_CONTINUE;
     }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info menuInfo) throws InterruptedException
     {
-        int menu = menuInfo.addRootMenu(menu_info_types.CONVERSE_START, null);
-        menu_info_data menuInfoData = menuInfo.getMenuItemById(menu);
-        menuInfoData.setServerNotify(false);
-        setCondition(self, CONDITION_CONVERSABLE);
-        faceTo(self, player);
+        clearCondition(self, CONDITION_CONVERSABLE);
         return SCRIPT_CONTINUE;
     }
     public int OnIncapacitated(obj_id self, obj_id killer) throws InterruptedException
@@ -65,30 +57,10 @@ public class chronicles_reward_vendor extends script.base_script
     }
     public int OnStartNpcConversation(obj_id self, obj_id player) throws InterruptedException
     {
-        obj_id npc = self;
-        if (ai_lib.isInCombat(npc) || ai_lib.isInCombat(player))
-        {
-            return SCRIPT_OVERRIDE;
-        }
-        if (chronicles_reward_vendor_condition__defaultCondition(player, npc))
-        {
-            chronicles_reward_vendor_action_showResourceVendorUI(player, npc);
-            string_id message = new string_id(c_stringFile, "s_4");
-            chat.chat(npc, player, message);
-            return SCRIPT_CONTINUE;
-        }
-        chat.chat(npc, "Error:  All conditions for OnStartNpcConversation were false.");
-        return SCRIPT_CONTINUE;
+        return SCRIPT_OVERRIDE;
     }
     public int OnNpcConversationResponse(obj_id self, String conversationId, obj_id player, string_id response) throws InterruptedException
     {
-        if (!conversationId.equals("chronicles_reward_vendor"))
-        {
-            return SCRIPT_CONTINUE;
-        }
-        obj_id npc = self;
-        int branchId = utils.getIntScriptVar(player, "conversation.chronicles_reward_vendor.branchId");
-        chat.chat(npc, "Error:  Fell through all branches and responses for OnNpcConversationResponse.");
         utils.removeScriptVar(player, "conversation.chronicles_reward_vendor.branchId");
         return SCRIPT_CONTINUE;
     }
diff --git a/sku.0/sys.server/compiled/game/script/conversation/storyteller_vendor.java b/sku.0/sys.server/compiled/game/script/conversation/storyteller_vendor.java
--- a/sku.0/sys.server/compiled/game/script/conversation/storyteller_vendor.java
+++ b/sku.0/sys.server/compiled/game/script/conversation/storyteller_vendor.java
@@ -75,24 +75,19 @@ public class storyteller_vendor extends script.base_script
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        if ((!isTangible(self)) || (isPlayer(self)))
-        {
-            detachScript(self, "conversation.storyteller_vendor");
-        }
-        setCondition(self, CONDITION_CONVERSABLE);
+        clearCondition(self, CONDITION_CONVERSABLE);
+        detachScript(self, "conversation.storyteller_vendor");
         return SCRIPT_CONTINUE;
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        setCondition(self, CONDITION_CONVERSABLE);
+        clearCondition(self, CONDITION_CONVERSABLE);
+        detachScript(self, "conversation.storyteller_vendor");
         return SCRIPT_CONTINUE;
     }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info menuInfo) throws InterruptedException
     {
-        int menu = menuInfo.addRootMenu(menu_info_types.CONVERSE_START, null);
-        menu_info_data menuInfoData = menuInfo.getMenuItemById(menu);
-        menuInfoData.setServerNotify(false);
-        setCondition(self, CONDITION_CONVERSABLE);
+        clearCondition(self, CONDITION_CONVERSABLE);
         return SCRIPT_CONTINUE;
     }
     public int OnIncapacitated(obj_id self, obj_id killer) throws InterruptedException
@@ -109,73 +104,10 @@ public class storyteller_vendor extends script.base_script
     }
     public int OnStartNpcConversation(obj_id self, obj_id player) throws InterruptedException
     {
-        obj_id npc = self;
-        if (ai_lib.isInCombat(npc) || ai_lib.isInCombat(player))
-        {
-            return SCRIPT_OVERRIDE;
-        }
-        if (storyteller_vendor_condition_checkFromToken(player, npc))
-        {
-            string_id message = new string_id(c_stringFile, "s_4");
-            int numberOfResponses = 0;
-            boolean hasResponse = false;
-            boolean hasResponse0 = false;
-            if (storyteller_vendor_condition__defaultCondition(player, npc))
-            {
-                ++numberOfResponses;
-                hasResponse = true;
-                hasResponse0 = true;
-            }
-            boolean hasResponse1 = false;
-            if (storyteller_vendor_condition__defaultCondition(player, npc))
-            {
-                ++numberOfResponses;
-                hasResponse = true;
-                hasResponse1 = true;
-            }
-            if (hasResponse)
-            {
-                int responseIndex = 0;
-                string_id responses[] = new string_id[numberOfResponses];
-                if (hasResponse0)
-                {
-                    responses[responseIndex++] = new string_id(c_stringFile, "s_6");
-                }
-                if (hasResponse1)
-                {
-                    responses[responseIndex++] = new string_id(c_stringFile, "s_10");
-                }
-                utils.setScriptVar(player, "conversation.storyteller_vendor.branchId", 1);
-                npcStartConversation(player, npc, "storyteller_vendor", message, responses);
-            }
-            else 
-            {
-                chat.chat(npc, player, message);
-            }
-            return SCRIPT_CONTINUE;
-        }
-        if (storyteller_vendor_condition__defaultCondition(player, npc))
-        {
-            string_id message = new string_id(c_stringFile, "s_14");
-            chat.chat(npc, player, message);
-            return SCRIPT_CONTINUE;
-        }
-        chat.chat(npc, "Error:  All conditions for OnStartNpcConversation were false.");
-        return SCRIPT_CONTINUE;
+        return SCRIPT_OVERRIDE;
     }
     public int OnNpcConversationResponse(obj_id self, String conversationId, obj_id player, string_id response) throws InterruptedException
     {
-        if (!conversationId.equals("storyteller_vendor"))
-        {
-            return SCRIPT_CONTINUE;
-        }
-        obj_id npc = self;
-        int branchId = utils.getIntScriptVar(player, "conversation.storyteller_vendor.branchId");
-        if (branchId == 1 && storyteller_vendor_handleBranch1(player, npc, response) == SCRIPT_CONTINUE)
-        {
-            return SCRIPT_CONTINUE;
-        }
-        chat.chat(npc, "Error:  Fell through all branches and responses for OnNpcConversationResponse.");
         utils.removeScriptVar(player, "conversation.storyteller_vendor.branchId");
         return SCRIPT_CONTINUE;
     }
diff --git a/sku.0/sys.server/compiled/game/script/conversation/fan_faire_pgc_c3po.java b/sku.0/sys.server/compiled/game/script/conversation/fan_faire_pgc_c3po.java
--- a/sku.0/sys.server/compiled/game/script/conversation/fan_faire_pgc_c3po.java
+++ b/sku.0/sys.server/compiled/game/script/conversation/fan_faire_pgc_c3po.java
@@ -15204,18 +15204,14 @@ public class fan_faire_pgc_c3po extends script.base_script
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        if ((!isMob(self)) || (isPlayer(self)))
-        {
-            detachScript(self, "conversation.fan_faire_pgc_c3po");
-        }
-        setCondition(self, CONDITION_CONVERSABLE);
-        messageTo(self, "handleFanFairec3poSetup", null, 2, false);
+        clearCondition(self, CONDITION_CONVERSABLE);
+        detachScript(self, "conversation.fan_faire_pgc_c3po");
         return SCRIPT_CONTINUE;
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        setCondition(self, CONDITION_CONVERSABLE);
-        messageTo(self, "handleFanFairec3poSetup", null, 2, false);
+        clearCondition(self, CONDITION_CONVERSABLE);
+        detachScript(self, "conversation.fan_faire_pgc_c3po");
         return SCRIPT_CONTINUE;
     }
     public int handleFanFairec3poSetup(obj_id self, dictionary params) throws InterruptedException
@@ -15230,10 +15226,7 @@ public class fan_faire_pgc_c3po extends script.base_script
     }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info menuInfo) throws InterruptedException
     {
-        int menu = menuInfo.addRootMenu(menu_info_types.CONVERSE_START, null);
-        menu_info_data menuInfoData = menuInfo.getMenuItemById(menu);
-        menuInfoData.setServerNotify(false);
-        faceTo(self, player);
+        clearCondition(self, CONDITION_CONVERSABLE);
         return SCRIPT_CONTINUE;
     }
     public int OnIncapacitated(obj_id self, obj_id killer) throws InterruptedException
@@ -15250,6 +15243,8 @@ public class fan_faire_pgc_c3po extends script.base_script
     }
     public int OnStartNpcConversation(obj_id self, obj_id player) throws InterruptedException
     {
+        return SCRIPT_OVERRIDE;
+        /*
         obj_id npc = self;
         if (ai_lib.isInCombat(npc) || ai_lib.isInCombat(player))
         {
@@ -15362,10 +15357,13 @@ public class fan_faire_pgc_c3po extends script.base_script
             return SCRIPT_CONTINUE;
         }
         chat.chat(npc, "Error:  All conditions for OnStartNpcConversation were false.");
-        return SCRIPT_CONTINUE;
+        */
     }
     public int OnNpcConversationResponse(obj_id self, String conversationId, obj_id player, string_id response) throws InterruptedException
     {
+        utils.removeScriptVar(player, "conversation.fan_faire_pgc_c3po.branchId");
+        return SCRIPT_CONTINUE;
+        /*
         if (!conversationId.equals("fan_faire_pgc_c3po"))
         {
             return SCRIPT_CONTINUE;
@@ -15442,6 +15440,6 @@ public class fan_faire_pgc_c3po extends script.base_script
         }
         chat.chat(npc, "Error:  Fell through all branches and responses for OnNpcConversationResponse.");
         utils.removeScriptVar(player, "conversation.fan_faire_pgc_c3po.branchId");
-        return SCRIPT_CONTINUE;
+        */
     }
 }
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/storyteller_vendor.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/storyteller_vendor.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/storyteller_vendor.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/storyteller_vendor.java
@@ -14,16 +14,14 @@ public class storyteller_vendor extends script.base_script
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        ai_lib.setDefaultCalmBehavior(self, ai_lib.BEHAVIOR_SENTINEL);
-        attachStorytellerVendorConvo(self);
-        messageTo(self, "handlSpecialName", null, 2, false);
+        detachScript(self, "conversation.storyteller_vendor");
+        detachScript(self, "systems.storyteller.storyteller_vendor");
         return SCRIPT_CONTINUE;
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        ai_lib.setDefaultCalmBehavior(self, ai_lib.BEHAVIOR_SENTINEL);
-        attachStorytellerVendorConvo(self);
-        messageTo(self, "handlSpecialName", null, 2, false);
+        detachScript(self, "conversation.storyteller_vendor");
+        detachScript(self, "systems.storyteller.storyteller_vendor");
         return SCRIPT_CONTINUE;
     }
     public int handlSpecialName(obj_id self, dictionary params) throws InterruptedException
@@ -65,35 +63,14 @@ public class storyteller_vendor extends script.base_script
     }
     public int msgStorytellerTokenTypeSelected(obj_id self, dictionary params) throws InterruptedException
     {
-        if ((params == null) || (params.isEmpty()))
-        {
-            return SCRIPT_CONTINUE;
-        }
-        obj_id player = sui.getPlayerId(params);
-        int btn = sui.getIntButtonPressed(params);
-        int idx = sui.getListboxSelectedRow(params);
-        String scriptvar_path = "storytellerVendor." + player;
-        String oldPidVar = scriptvar_path + ".pid";
-        if (utils.hasScriptVar(self, oldPidVar))
-        {
-            utils.removeScriptVar(self, oldPidVar);
-        }
-        if (btn == sui.BP_CANCEL)
-        {
-            return SCRIPT_CONTINUE;
-        }
-        int tokenType = idx;
-        storyteller.displayStorytellerTokenPurchaseSUI(player, tokenType, self);
         return SCRIPT_CONTINUE;
     }
     public int msgStorytellerTokenPurchaseSelected(obj_id self, dictionary params) throws InterruptedException
     {
-        storyteller.storytellerTokenPurchased(params, self);
         return SCRIPT_CONTINUE;
     }
     public int msgStorytellerChargesSelected(obj_id self, dictionary params) throws InterruptedException
     {
-        storyteller.storytellerSellTokenWithCharges(params, self);
         return SCRIPT_CONTINUE;
     }
 }
