diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint.java
@@ -25,12 +25,12 @@ public class blueprint extends script.base_script
     };
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        messageTo(self, "handleStorytellerBlueprintInitialize", null, 1, false);
+        detachScript(self, "systems.storyteller.blueprint");
         return SCRIPT_CONTINUE;
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        messageTo(self, "handleStorytellerBlueprintInitialize", null, 5, false);
+        detachScript(self, "systems.storyteller.blueprint");
         return SCRIPT_CONTINUE;
     }
     public int handleStorytellerBlueprintInitialize(obj_id self, dictionary params) throws InterruptedException
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint_blank.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint_blank.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint_blank.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/blueprint_blank.java
@@ -11,6 +11,16 @@ public class blueprint_blank extends script.base_script
     public blueprint_blank()
     {
     }
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.blueprint_blank");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.blueprint_blank");
+        return SCRIPT_CONTINUE;
+    }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info mi) throws InterruptedException
     {
         mi.addRootMenu(menu_info_types.ITEM_USE, new string_id("storyteller", "blueprint_record_data"));
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/destructible_prop_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/destructible_prop_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/destructible_prop_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/destructible_prop_token.java
@@ -12,6 +12,16 @@ public class destructible_prop_token extends script.base_script
     }
     public static final int MIN_HEALTH = 180000;
     public static final int MAX_HEALTH = 250000;
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.destructible_prop_token");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.destructible_prop_token");
+        return SCRIPT_CONTINUE;
+    }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info mi) throws InterruptedException
     {
         if (!utils.isNestedWithin(self, player))
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/effect_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/effect_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/effect_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/effect_token.java
@@ -12,12 +12,12 @@ public class effect_token extends script.base_script
     public static final String EFFECT_CONTROL_SCRIPT = "systems.storyteller.effect_controller";
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        storyteller.resetTokenDailyCount(self);
+        detachScript(self, "systems.storyteller.effect_token");
         return SCRIPT_CONTINUE;
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        storyteller.resetTokenDailyCount(self);
+        detachScript(self, "systems.storyteller.effect_token");
         return SCRIPT_CONTINUE;
     }
     public int storytellerEffectTokenDailyAlarm(obj_id self, dictionary params) throws InterruptedException
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/jukebox_converter_prop_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/jukebox_converter_prop_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/jukebox_converter_prop_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/jukebox_converter_prop_token.java
@@ -11,6 +11,16 @@ public class jukebox_converter_prop_token extends script.base_script
     {
     }
     public static final String JUKEBOX_SCRIPT = "systems.event_perk.jukebox";
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.jukebox_converter_prop_token");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.jukebox_converter_prop_token");
+        return SCRIPT_CONTINUE;
+    }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info mi) throws InterruptedException
     {
         if (!utils.isNestedWithin(self, player))
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_difficulty_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_difficulty_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_difficulty_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_difficulty_token.java
@@ -11,6 +11,16 @@ public class npc_difficulty_token extends script.base_script
     public npc_difficulty_token()
     {
     }
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.npc_difficulty_token");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.npc_difficulty_token");
+        return SCRIPT_CONTINUE;
+    }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info mi) throws InterruptedException
     {
         if (!utils.isNestedWithin(self, player))
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/npc_token.java
@@ -13,12 +13,12 @@ public class npc_token extends script.base_script
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        storyteller.resetTokenDailyCount(self);
+        detachScript(self, "systems.storyteller.npc_token");
         return SCRIPT_CONTINUE;
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        storyteller.resetTokenDailyCount(self);
+        detachScript(self, "systems.storyteller.npc_token");
         return SCRIPT_CONTINUE;
     }
     public int storytellerEffectTokenDailyAlarm(obj_id self, dictionary params) throws InterruptedException
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/prop_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/prop_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/prop_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/prop_token.java
@@ -11,12 +11,12 @@ public class prop_token extends script.base_script
     }
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        storyteller.resetTokenDailyCount(self);
+        detachScript(self, "systems.storyteller.prop_token");
         return SCRIPT_CONTINUE;
     }
     public int OnInitialize(obj_id self) throws InterruptedException
     {
-        storyteller.resetTokenDailyCount(self);
+        detachScript(self, "systems.storyteller.prop_token");
         return SCRIPT_CONTINUE;
     }
     public int storytellerEffectTokenDailyAlarm(obj_id self, dictionary params) throws InterruptedException
diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/theater_token.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/theater_token.java
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/theater_token.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/theater_token.java
@@ -11,6 +11,16 @@ public class theater_token extends script.base_script
     public theater_token()
     {
     }
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.theater_token");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.theater_token");
+        return SCRIPT_CONTINUE;
+    }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info mi) throws InterruptedException
     {
         mi.addRootMenu(menu_info_types.ITEM_USE, new string_id("storyteller", "deploy_mode"));
