diff --git a/sku.0/sys.server/compiled/game/script/systems/storyteller/invitation_terminal.java b/sku.0/sys.server/compiled/game/script/systems/storyteller/invitation_terminal.java
index ab908bf..3f2ea64 100644
--- a/sku.0/sys.server/compiled/game/script/systems/storyteller/invitation_terminal.java
+++ b/sku.0/sys.server/compiled/game/script/systems/storyteller/invitation_terminal.java
@@ -13,7 +13,12 @@ public class invitation_terminal extends script.base_script
     public static final String JUKEBOX_SCRIPT = "systems.event_perk.jukebox";
     public int OnAttach(obj_id self) throws InterruptedException
     {
-        messageTo(self, "initializeStorytellerInvitationTerminal", null, 1, false);
+        detachScript(self, "systems.storyteller.invitation_terminal");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "systems.storyteller.invitation_terminal");
         return SCRIPT_CONTINUE;
     }
     public int initializeStorytellerInvitationTerminal(obj_id self, dictionary params) throws InterruptedException
