diff --git a/sku.0/sys.server/compiled/game/script/event/ewok_festival/fountain.java b/sku.0/sys.server/compiled/game/script/event/ewok_festival/fountain.java
index 463858d..1f5ad46 100644
--- a/sku.0/sys.server/compiled/game/script/event/ewok_festival/fountain.java
+++ b/sku.0/sys.server/compiled/game/script/event/ewok_festival/fountain.java
@@ -18,6 +18,16 @@ public class fountain extends script.base_script
     public static final string_id BERRY_GRANTED = new string_id("spam", "berry_granted");
     public static final String OBJ_BERRY_ONE = "item_event_ewok_berry_01_01";
     public static final String OBJ_BERRY_TWO = "item_event_ewok_berry_01_02";
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "event.ewok_festival.fountain");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "event.ewok_festival.fountain");
+        return SCRIPT_CONTINUE;
+    }
     public int OnObjectMenuRequest(obj_id self, obj_id player, menu_info mi) throws InterruptedException
     {
         if (utils.playerHasStaticItemInBankOrInventory(player, OBJ_BERRY_ONE))
diff --git a/sku.0/sys.server/compiled/game/script/event/lifeday/lifeday_tree.java b/sku.0/sys.server/compiled/game/script/event/lifeday/lifeday_tree.java
index af7dc32..ac56413 100644
--- a/sku.0/sys.server/compiled/game/script/event/lifeday/lifeday_tree.java
+++ b/sku.0/sys.server/compiled/game/script/event/lifeday/lifeday_tree.java
@@ -23,6 +23,16 @@ public class lifeday_tree extends script.base_script
     private static final String LIFEDAY_BADGE = "lifeday_badge_11";
     private static final string_id TREE_BADGE = new string_id("spam", "tree_badge");
 
+    public int OnAttach(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "event.lifeday.lifeday_tree");
+        return SCRIPT_CONTINUE;
+    }
+    public int OnInitialize(obj_id self) throws InterruptedException
+    {
+        detachScript(self, "event.lifeday.lifeday_tree");
+        return SCRIPT_CONTINUE;
+    }
     private String currentYearObjVar() throws InterruptedException
     {
         return utils.XMAS_RECEIVED_IX_01;
