diff --git a/sku.0/sys.server/compiled/game/script/library/spawning.java b/sku.0/sys.server/compiled/game/script/library/spawning.java
index cd630a4..3c1229d 100644
--- a/sku.0/sys.server/compiled/game/script/library/spawning.java
+++ b/sku.0/sys.server/compiled/game/script/library/spawning.java
@@ -293,7 +293,9 @@ public class spawning extends script.base_script
         {
             return false;
         }
-        if (hasObjVar(spawner, "eventRequired") && getStringObjVar(spawner, "eventRequired").equals("loveday"))
+        if (hasObjVar(spawner, "eventRequired") &&
+            (getStringObjVar(spawner, "eventRequired").equals("loveday") ||
+            getStringObjVar(spawner, "eventRequired").equals("life_day")))
         {
             return true;
         }
