diff --git a/sku.0/sys.server/compiled/game/script/library/gcw.java b/sku.0/sys.server/compiled/game/script/library/gcw.java
index 0c49dfc..8799fe0 100755
--- a/sku.0/sys.server/compiled/game/script/library/gcw.java
+++ b/sku.0/sys.server/compiled/game/script/library/gcw.java
@@ -2691,10 +2691,6 @@ public class gcw extends script.base_script
             CustomerServiceLog("gcw_city_invasion", "gcw.gcwIsInvasionCityOn: GCW City: " + city + " is not configured to run a city invasion. Function returning False.");
             return false;
         }
-        if (city.equalsIgnoreCase("dearic") && utils.checkConfigFlag("GameServer", "lifeday")) {
-            CustomerServiceLog("gcw_city_invasion", "gcw.gcwIsInvasionCityOn: GCW City: " + city + " is not running because life day is turned on.");
-            return false;
-        }
         return true;
     }
     public static int gcwGetNextInvasionHour(String cityName) throws InterruptedException
