diff --git a/sku.0/sys.server/compiled/game/script/library/space_combat.java b/sku.0/sys.server/compiled/game/script/library/space_combat.java
index b7fdc454a..228804e30 100755
--- a/sku.0/sys.server/compiled/game/script/library/space_combat.java
+++ b/sku.0/sys.server/compiled/game/script/library/space_combat.java
@@ -999,15 +999,7 @@ public class space_combat extends script.base_script
             if (strItems.length > 0)
             {
                 int intRoll2 = rand(0, strItems.length - 1);
-                String itemTemplateName;
-                if (getConfigSetting("GameServer", "enableLevelUpLoot") != null && (rand(1, 10000) == 1))
-                {
-                    itemTemplateName = "object/tangible/loot/quest/levelup_lifeday_orb.iff";
-                }
-                else 
-                {
-                    itemTemplateName = strItems[intRoll2];
-                }
+                String itemTemplateName = strItems[intRoll2];
                 if (space_battlefield.isInBattlefield(objAttacker))
                 {
                     CustomerServiceLog("battlefield", "%TU Created " + itemTemplateName + " in " + objContainer + " contained by " + objAttacker, getOwner(objAttacker));
