diff --git a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
index 21fe70f..a076357 100644
--- a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
@@ -1936,3 +1936,6 @@ headShot3	combat	normal	headShot3	failSpecialAttack			2	headShot3	1	0	1	1	1	0	1
 melee1hBodyHit1	combat	normal	melee1hBodyHit1	failSpecialAttack			1.5	melee1hBodyHit1	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	1HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
 melee1hBodyHit2	combat	normal	melee1hBodyHit2	failSpecialAttack			2	melee1hBodyHit2	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	1HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
 melee1hBodyHit3	combat	normal	melee1hBodyHit3	failSpecialAttack			2.25	melee1hBodyHit3	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	1HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
+melee2hHeadHit1	combat	normal	melee2hHeadHit1	failSpecialAttack			1.25	melee2hHeadHit1	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	2HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
+melee2hHeadHit2	combat	normal	melee2hHeadHit2	failSpecialAttack			1.75	melee2hHeadHit2	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	2HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
+melee2hHeadHit3	combat	normal	melee2hHeadHit3	failSpecialAttack			2.25	melee2hHeadHit3	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	2HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
index ec153ba..68b7895 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
@@ -1300,3 +1300,6 @@ headShot3	headShot3			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/
 melee1hBodyHit1	melee1hBodyHit1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	counter_high_right				counter_high_right										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	1.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				1HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
 melee1hBodyHit2	melee1hBodyHit2			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	combo_4b				combo_4b										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	2.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				1HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
 melee1hBodyHit3	melee1hBodyHit3			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	combo_3a				combo_3a										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	3.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				1HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
+melee2hHeadHit1	melee2hHeadHit1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	combo_2d				combo_2d										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	1.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				2HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
+melee2hHeadHit2	melee2hHeadHit2			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	combo_2d				combo_2d										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	2.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				2HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
+melee2hHeadHit3	melee2hHeadHit3			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	combo_2d				combo_2d										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	3.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				2HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
index 21b5ff9..336208f 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
@@ -10,6 +10,9 @@ headShot3	0.5	0.5	2.5	MIND	2	5	RANGED
 melee1hBodyHit1	0.5	0.5	0.625	HEALTH	1.5	25	INTENSITY
 melee1hBodyHit2	0.75	0.75	1.25	HEALTH	2	25	INTENSITY
 melee1hBodyHit3	1	1	2	HEALTH	2.25	25	INTENSITY
+melee2hHeadHit1	0.5	1	0.5	MIND	1.25	10	INTENSITY
+melee2hHeadHit2	1	1.5	1	MIND	1.75	10	INTENSITY
+melee2hHeadHit3	1.5	2	1.5	MIND	2.25	10	INTENSITY
 legShot1	2.25	1.25	0.5	ACTION	2	25	NONE
 polearmLegHit1	1	0.5	0.5	ACTION	1.25	10	NONE
 polearmSpinAttack1	1.5	1	1	RANDOM	1.5	10	NONE
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
index c7c81c3..34d63be 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
@@ -9,6 +9,9 @@ headShot3	masterheadshot
 melee1hBodyHit1	saimai
 melee1hBodyHit2	saisun
 melee1hBodyHit3	saitok
+melee2hHeadHit1	scalpblow
+melee2hHeadHit2	scalpstrike
+melee2hHeadHit3	scalpslam
 legShot1	leg
 polearmLegHit1	legbruiser
 polearmSpinAttack1	limbsmasher
diff --git a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
index 431c733..2b8718f 100644
--- a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
+++ b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
@@ -14663,4 +14663,28 @@ public class combat_actions extends script.systems.combat.combat_base
         }
         return SCRIPT_CONTINUE;
     }
+    public int melee2hHeadHit1(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
+    {
+        if (!combatStandardAction("melee2hHeadHit1", self, target, params, "", ""))
+        {
+            return SCRIPT_OVERRIDE;
+        }
+        return SCRIPT_CONTINUE;
+    }
+    public int melee2hHeadHit2(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
+    {
+        if (!combatStandardAction("melee2hHeadHit2", self, target, params, "", ""))
+        {
+            return SCRIPT_OVERRIDE;
+        }
+        return SCRIPT_CONTINUE;
+    }
+    public int melee2hHeadHit3(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
+    {
+        if (!combatStandardAction("melee2hHeadHit3", self, target, params, "", ""))
+        {
+            return SCRIPT_OVERRIDE;
+        }
+        return SCRIPT_CONTINUE;
+    }
 }
diff --git a/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java b/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
index 1db2f65..20b4ff0 100644
--- a/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
+++ b/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
@@ -32,6 +32,9 @@ public class precu_headshot1_fixture extends script.base_script
     private static final String ONE_HAND_BODY_TWO_COMMAND = "melee1hBodyHit2";
     private static final String ONE_HAND_BODY_THREE_COMMAND = "melee1hBodyHit3";
     private static final String TWO_HAND_AREA_COMMAND = "melee2hSpinAttack1";
+    private static final String TWO_HAND_HEAD_ONE_COMMAND = "melee2hHeadHit1";
+    private static final String TWO_HAND_HEAD_TWO_COMMAND = "melee2hHeadHit2";
+    private static final String TWO_HAND_HEAD_THREE_COMMAND = "melee2hHeadHit3";
     private static final String POLEARM_CERTIFICATION = "cert_lance_staff_wood_s2";
     private static final String ONE_HAND_CERTIFICATION = "cert_sword_blade_rantok";
     private static final String TWO_HAND_CERTIFICATION = "cert_sword_2h_cleaver";
@@ -87,6 +90,12 @@ public class precu_headshot1_fixture extends script.base_script
         ROOT + ".originalOneHandBodyThreeCommand";
     private static final String ORIGINAL_TWO_HAND_AREA_COMMAND =
         ROOT + ".originalTwoHandAreaCommand";
+    private static final String ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND =
+        ROOT + ".originalTwoHandHeadOneCommand";
+    private static final String ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND =
+        ROOT + ".originalTwoHandHeadTwoCommand";
+    private static final String ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND =
+        ROOT + ".originalTwoHandHeadThreeCommand";
     private static final String ORIGINAL_POLEARM_CERTIFICATION =
         ROOT + ".originalPolearmCertification";
     private static final String ORIGINAL_ONE_HAND_CERTIFICATION =
@@ -272,6 +281,15 @@ public class precu_headshot1_fixture extends script.base_script
         boolean twoHandAreaCommandGranted =
             hasCommand(attacker, TWO_HAND_AREA_COMMAND) ||
             grantCommand(attacker, TWO_HAND_AREA_COMMAND);
+        boolean twoHandHeadOneCommandGranted =
+            hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND) ||
+            grantCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND);
+        boolean twoHandHeadTwoCommandGranted =
+            hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND) ||
+            grantCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND);
+        boolean twoHandHeadThreeCommandGranted =
+            hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND) ||
+            grantCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND);
         boolean polearmCertificationGranted =
             hasCommand(attacker, POLEARM_CERTIFICATION) ||
             grantCommand(attacker, POLEARM_CERTIFICATION);
@@ -303,6 +321,12 @@ public class precu_headshot1_fixture extends script.base_script
             !hasCommand(attacker, ONE_HAND_BODY_THREE_COMMAND) ||
             !twoHandAreaCommandGranted ||
             !hasCommand(attacker, TWO_HAND_AREA_COMMAND) ||
+            !twoHandHeadOneCommandGranted ||
+            !hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND) ||
+            !twoHandHeadTwoCommandGranted ||
+            !hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND) ||
+            !twoHandHeadThreeCommandGranted ||
+            !hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND) ||
             !polearmCertificationGranted ||
             !hasCommand(attacker, POLEARM_CERTIFICATION) ||
             !oneHandCertificationGranted ||
@@ -334,6 +358,12 @@ public class precu_headshot1_fixture extends script.base_script
                     hasCommand(attacker, ONE_HAND_BODY_THREE_COMMAND) +
                 " twoHandAreaCommand=" +
                     hasCommand(attacker, TWO_HAND_AREA_COMMAND) +
+                " twoHandHeadOneCommand=" +
+                    hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND) +
+                " twoHandHeadTwoCommand=" +
+                    hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND) +
+                " twoHandHeadThreeCommand=" +
+                    hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND) +
                 " polearmCertification=" +
                     hasCommand(attacker, POLEARM_CERTIFICATION) +
                 " oneHandCertification=" +
@@ -553,6 +583,24 @@ public class precu_headshot1_fixture extends script.base_script
         {
             revokeCommand(attacker, TWO_HAND_AREA_COMMAND);
         }
+        if (attackerRoot && hasObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND) == 0 &&
+            hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND))
+        {
+            revokeCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND);
+        }
+        if (attackerRoot && hasObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND) == 0 &&
+            hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND))
+        {
+            revokeCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND);
+        }
+        if (attackerRoot && hasObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND) == 0 &&
+            hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND))
+        {
+            revokeCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND);
+        }
         if (attackerRoot && hasObjVar(attacker, ORIGINAL_POLEARM_CERTIFICATION) &&
             getIntObjVar(attacker, ORIGINAL_POLEARM_CERTIFICATION) == 0 &&
             hasCommand(attacker, POLEARM_CERTIFICATION))
@@ -777,6 +825,12 @@ public class precu_headshot1_fixture extends script.base_script
             hasCommand(player, ONE_HAND_BODY_THREE_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_TWO_HAND_AREA_COMMAND,
             hasCommand(player, TWO_HAND_AREA_COMMAND) ? 1 : 0);
+        setObjVar(player, ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND,
+            hasCommand(player, TWO_HAND_HEAD_ONE_COMMAND) ? 1 : 0);
+        setObjVar(player, ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND,
+            hasCommand(player, TWO_HAND_HEAD_TWO_COMMAND) ? 1 : 0);
+        setObjVar(player, ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND,
+            hasCommand(player, TWO_HAND_HEAD_THREE_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_POLEARM_CERTIFICATION,
             hasCommand(player, POLEARM_CERTIFICATION) ? 1 : 0);
         setObjVar(player, ORIGINAL_ONE_HAND_CERTIFICATION,
@@ -894,6 +948,24 @@ public class precu_headshot1_fixture extends script.base_script
         {
             revokeCommand(attacker, TWO_HAND_AREA_COMMAND);
         }
+        if (hasObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND) == 0 &&
+            hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND))
+        {
+            revokeCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND);
+        }
+        if (hasObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND) == 0 &&
+            hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND))
+        {
+            revokeCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND);
+        }
+        if (hasObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND) == 0 &&
+            hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND))
+        {
+            revokeCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND);
+        }
         if (hasObjVar(attacker, ORIGINAL_POLEARM_CERTIFICATION) &&
             getIntObjVar(attacker, ORIGINAL_POLEARM_CERTIFICATION) == 0 &&
             hasCommand(attacker, POLEARM_CERTIFICATION))
@@ -1067,6 +1139,15 @@ public class precu_headshot1_fixture extends script.base_script
         boolean twoHandAreaCommandReady =
             hasCommand(attacker, TWO_HAND_AREA_COMMAND) ||
             grantCommand(attacker, TWO_HAND_AREA_COMMAND);
+        boolean twoHandHeadOneCommandReady =
+            hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND) ||
+            grantCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND);
+        boolean twoHandHeadTwoCommandReady =
+            hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND) ||
+            grantCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND);
+        boolean twoHandHeadThreeCommandReady =
+            hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND) ||
+            grantCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND);
         boolean polearmCertificationReady =
             hasCommand(attacker, POLEARM_CERTIFICATION) ||
             grantCommand(attacker, POLEARM_CERTIFICATION);
@@ -1083,7 +1164,9 @@ public class precu_headshot1_fixture extends script.base_script
             polearmAreaCommandReady && oneHandAreaCommandReady &&
             oneHandBodyOneCommandReady &&
             oneHandBodyTwoCommandReady && oneHandBodyThreeCommandReady &&
-            twoHandAreaCommandReady && polearmCertificationReady &&
+            twoHandAreaCommandReady && twoHandHeadOneCommandReady &&
+            twoHandHeadTwoCommandReady && twoHandHeadThreeCommandReady &&
+            polearmCertificationReady &&
             oneHandCertificationReady && twoHandCertificationReady &&
             attackerStanding && defenderStanding &&
             attackerStationary && defenderStationary &&
@@ -1249,6 +1332,9 @@ public class precu_headshot1_fixture extends script.base_script
             hasObjVar(player, ORIGINAL_ONE_HAND_BODY_TWO_COMMAND) &&
             hasObjVar(player, ORIGINAL_ONE_HAND_BODY_THREE_COMMAND) &&
             hasObjVar(player, ORIGINAL_TWO_HAND_AREA_COMMAND) &&
+            hasObjVar(player, ORIGINAL_TWO_HAND_HEAD_ONE_COMMAND) &&
+            hasObjVar(player, ORIGINAL_TWO_HAND_HEAD_TWO_COMMAND) &&
+            hasObjVar(player, ORIGINAL_TWO_HAND_HEAD_THREE_COMMAND) &&
             hasObjVar(player, ORIGINAL_POLEARM_CERTIFICATION) &&
             hasObjVar(player, ORIGINAL_ONE_HAND_CERTIFICATION) &&
             hasObjVar(player, ORIGINAL_TWO_HAND_CERTIFICATION) &&
@@ -1332,6 +1418,12 @@ public class precu_headshot1_fixture extends script.base_script
             combat.canPerformAction(ONE_HAND_BODY_THREE_COMMAND, attacker) : -1;
         int canPerformTwoHandArea = isIdValid(weapon) ?
             combat.canPerformAction(TWO_HAND_AREA_COMMAND, attacker) : -1;
+        int canPerformTwoHandHeadOne = isIdValid(weapon) ?
+            combat.canPerformAction(TWO_HAND_HEAD_ONE_COMMAND, attacker) : -1;
+        int canPerformTwoHandHeadTwo = isIdValid(weapon) ?
+            combat.canPerformAction(TWO_HAND_HEAD_TWO_COMMAND, attacker) : -1;
+        int canPerformTwoHandHeadThree = isIdValid(weapon) ?
+            combat.canPerformAction(TWO_HAND_HEAD_THREE_COMMAND, attacker) : -1;
         String fixtureWeapon = hasObjVar(attacker, FIXTURE_WEAPON) ?
             getObjIdObjVar(attacker, FIXTURE_WEAPON).toString() : "none";
         String fixturePistol = hasObjVar(attacker, FIXTURE_PISTOL) ?
@@ -1376,6 +1468,12 @@ public class precu_headshot1_fixture extends script.base_script
                 hasCommand(attacker, ONE_HAND_BODY_THREE_COMMAND) +
             " twoHandAreaCommand=" +
                 hasCommand(attacker, TWO_HAND_AREA_COMMAND) +
+            " twoHandHeadOneCommand=" +
+                hasCommand(attacker, TWO_HAND_HEAD_ONE_COMMAND) +
+            " twoHandHeadTwoCommand=" +
+                hasCommand(attacker, TWO_HAND_HEAD_TWO_COMMAND) +
+            " twoHandHeadThreeCommand=" +
+                hasCommand(attacker, TWO_HAND_HEAD_THREE_COMMAND) +
             " polearmCertification=" +
                 hasCommand(attacker, POLEARM_CERTIFICATION) +
             " oneHandCertification=" +
@@ -1397,6 +1495,9 @@ public class precu_headshot1_fixture extends script.base_script
             " canPerformOneHandBodyTwo=" + canPerformOneHandBodyTwo +
             " canPerformOneHandBodyThree=" + canPerformOneHandBodyThree +
             " canPerformTwoHandArea=" + canPerformTwoHandArea +
+            " canPerformTwoHandHeadOne=" + canPerformTwoHandHeadOne +
+            " canPerformTwoHandHeadTwo=" + canPerformTwoHandHeadTwo +
+            " canPerformTwoHandHeadThree=" + canPerformTwoHandHeadThree +
             " fixtureWeapon=" + fixtureWeapon +
             " fixturePistol=" + fixturePistol +
             " fixturePolearm=" + fixturePolearm +
