diff --git a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
index 1c9943a..10c5da0 100644
--- a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
+++ b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
@@ -14712,6 +14712,16 @@ public class combat_actions extends script.systems.combat.combat_base
         }
         return SCRIPT_CONTINUE;
     }
+    public int fullAutoArea1(obj_id self, obj_id target, String params,
+        float defaultTime) throws InterruptedException
+    {
+        if (!combatStandardAction(
+            "fullAutoArea1", self, target, params, "", ""))
+        {
+            return SCRIPT_OVERRIDE;
+        }
+        return SCRIPT_CONTINUE;
+    }
     public int scatterShot1(obj_id self, obj_id target, String params,
         float defaultTime) throws InterruptedException
     {
diff --git a/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java b/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
index 3808ac8..a7e0de4 100644
--- a/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
+++ b/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
@@ -42,6 +42,8 @@ public class precu_headshot1_fixture extends script.base_script
         "combat_carbine_accuracy_03";
     private static final String CARBINE_ACCURACY_FOUR =
         "combat_carbine_accuracy_04";
+    private static final String CARBINE_SUPPORT_ONE =
+        "combat_carbine_support_01";
     private static final String CARBINE_SPEED_ONE = "combat_carbine_speed_01";
     private static final String CARBINE_ABILITY_ONE =
         "combat_carbine_ability_01";
@@ -95,6 +97,8 @@ public class precu_headshot1_fixture extends script.base_script
         "fullAutoSingle1";
     private static final String FULL_AUTO_SINGLE_TWO_COMMAND =
         "fullAutoSingle2";
+    private static final String FULL_AUTO_AREA_ONE_COMMAND =
+        "fullAutoArea1";
     private static final String SCATTER_SHOT_ONE_COMMAND = "scatterShot1";
     private static final String SCATTER_SHOT_TWO_COMMAND = "scatterShot2";
     private static final String WILD_SHOT_ONE_COMMAND = "wildShot1";
@@ -186,6 +190,8 @@ public class precu_headshot1_fixture extends script.base_script
         ROOT + ".originalCarbineAccuracyThree";
     private static final String ORIGINAL_CARBINE_ACCURACY_FOUR =
         ROOT + ".originalCarbineAccuracyFour";
+    private static final String ORIGINAL_CARBINE_SUPPORT_ONE =
+        ROOT + ".originalCarbineSupportOne";
     private static final String ORIGINAL_CARBINE_SPEED_ONE =
         ROOT + ".originalCarbineSpeedOne";
     private static final String ORIGINAL_CARBINE_ABILITY_ONE =
@@ -274,6 +280,8 @@ public class precu_headshot1_fixture extends script.base_script
         ROOT + ".originalFullAutoSingleOneCommand";
     private static final String ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND =
         ROOT + ".originalFullAutoSingleTwoCommand";
+    private static final String ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND =
+        ROOT + ".originalFullAutoAreaOneCommand";
     private static final String ORIGINAL_SCATTER_SHOT_ONE_COMMAND =
         ROOT + ".originalScatterShotOneCommand";
     private static final String ORIGINAL_SCATTER_SHOT_TWO_COMMAND =
@@ -564,6 +572,9 @@ public class precu_headshot1_fixture extends script.base_script
         boolean carbineAccuracyFourGranted = carbineAccuracyThreeGranted &&
             (hasSkill(attacker, CARBINE_ACCURACY_FOUR) ||
                 grantSkill(attacker, CARBINE_ACCURACY_FOUR));
+        boolean carbineSupportOneGranted = carbineNoviceGranted &&
+            (hasSkill(attacker, CARBINE_SUPPORT_ONE) ||
+                grantSkill(attacker, CARBINE_SUPPORT_ONE));
         boolean carbineSpeedOneGranted = carbineNoviceGranted &&
             (hasSkill(attacker, CARBINE_SPEED_ONE) ||
                 grantSkill(attacker, CARBINE_SPEED_ONE));
@@ -686,6 +697,9 @@ public class precu_headshot1_fixture extends script.base_script
         boolean fullAutoSingleTwoCommandGranted =
             hasCommand(attacker, FULL_AUTO_SINGLE_TWO_COMMAND) ||
             grantCommand(attacker, FULL_AUTO_SINGLE_TWO_COMMAND);
+        boolean fullAutoAreaOneCommandGranted =
+            hasCommand(attacker, FULL_AUTO_AREA_ONE_COMMAND) ||
+            grantCommand(attacker, FULL_AUTO_AREA_ONE_COMMAND);
         boolean scatterShotOneCommandGranted =
             hasCommand(attacker, SCATTER_SHOT_ONE_COMMAND) ||
             grantCommand(attacker, SCATTER_SHOT_ONE_COMMAND);
@@ -806,6 +820,7 @@ public class precu_headshot1_fixture extends script.base_script
             !carbineNoviceGranted || !carbineAccuracyOneGranted ||
             !carbineAccuracyTwoGranted || !carbineAccuracyThreeGranted ||
             !carbineAccuracyFourGranted ||
+            !carbineSupportOneGranted ||
             !carbineSpeedOneGranted ||
             !carbineAbilityFourGranted ||
             !pistolOneGranted ||
@@ -875,6 +890,8 @@ public class precu_headshot1_fixture extends script.base_script
             !hasCommand(attacker, FULL_AUTO_SINGLE_ONE_COMMAND) ||
             !fullAutoSingleTwoCommandGranted ||
             !hasCommand(attacker, FULL_AUTO_SINGLE_TWO_COMMAND) ||
+            !fullAutoAreaOneCommandGranted ||
+            !hasCommand(attacker, FULL_AUTO_AREA_ONE_COMMAND) ||
             !scatterShotOneCommandGranted ||
             !hasCommand(attacker, SCATTER_SHOT_ONE_COMMAND) ||
             !scatterShotTwoCommandGranted ||
@@ -1456,6 +1473,8 @@ public class precu_headshot1_fixture extends script.base_script
                 FULL_AUTO_SINGLE_ONE_COMMAND);
             restoreCommand(attacker, ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND,
                 FULL_AUTO_SINGLE_TWO_COMMAND);
+            restoreCommand(attacker, ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND,
+                FULL_AUTO_AREA_ONE_COMMAND);
             restoreCommand(attacker, ORIGINAL_SCATTER_SHOT_ONE_COMMAND,
                 SCATTER_SHOT_ONE_COMMAND);
             restoreCommand(attacker, ORIGINAL_SCATTER_SHOT_TWO_COMMAND,
@@ -1687,6 +1706,13 @@ public class precu_headshot1_fixture extends script.base_script
         {
             revokeSkill(attacker, CARBINE_ACCURACY_ONE);
         }
+        if (attackerRoot &&
+            hasObjVar(attacker, ORIGINAL_CARBINE_SUPPORT_ONE) &&
+            getIntObjVar(attacker, ORIGINAL_CARBINE_SUPPORT_ONE) == 0 &&
+            hasSkill(attacker, CARBINE_SUPPORT_ONE))
+        {
+            revokeSkill(attacker, CARBINE_SUPPORT_ONE);
+        }
         if (attackerRoot && hasObjVar(attacker, ORIGINAL_CARBINE_NOVICE) &&
             getIntObjVar(attacker, ORIGINAL_CARBINE_NOVICE) == 0 &&
             hasSkill(attacker, CARBINE_NOVICE))
@@ -2074,6 +2100,8 @@ public class precu_headshot1_fixture extends script.base_script
             hasSkill(player, CARBINE_ACCURACY_THREE) ? 1 : 0);
         setObjVar(player, ORIGINAL_CARBINE_ACCURACY_FOUR,
             hasSkill(player, CARBINE_ACCURACY_FOUR) ? 1 : 0);
+        setObjVar(player, ORIGINAL_CARBINE_SUPPORT_ONE,
+            hasSkill(player, CARBINE_SUPPORT_ONE) ? 1 : 0);
         setObjVar(player, ORIGINAL_CARBINE_SPEED_ONE,
             hasSkill(player, CARBINE_SPEED_ONE) ? 1 : 0);
         setObjVar(player, ORIGINAL_CARBINE_ABILITY_ONE,
@@ -2166,6 +2194,8 @@ public class precu_headshot1_fixture extends script.base_script
             hasCommand(player, FULL_AUTO_SINGLE_ONE_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND,
             hasCommand(player, FULL_AUTO_SINGLE_TWO_COMMAND) ? 1 : 0);
+        setObjVar(player, ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND,
+            hasCommand(player, FULL_AUTO_AREA_ONE_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_SCATTER_SHOT_ONE_COMMAND,
             hasCommand(player, SCATTER_SHOT_ONE_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_SCATTER_SHOT_TWO_COMMAND,
@@ -2487,6 +2517,8 @@ public class precu_headshot1_fixture extends script.base_script
             FULL_AUTO_SINGLE_ONE_COMMAND);
         restoreCommand(attacker, ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND,
             FULL_AUTO_SINGLE_TWO_COMMAND);
+        restoreCommand(attacker, ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND,
+            FULL_AUTO_AREA_ONE_COMMAND);
         restoreCommand(attacker, ORIGINAL_SCATTER_SHOT_ONE_COMMAND,
             SCATTER_SHOT_ONE_COMMAND);
         restoreCommand(attacker, ORIGINAL_SCATTER_SHOT_TWO_COMMAND,
@@ -2701,6 +2733,12 @@ public class precu_headshot1_fixture extends script.base_script
         {
             revokeSkill(attacker, CARBINE_ACCURACY_ONE);
         }
+        if (hasObjVar(attacker, ORIGINAL_CARBINE_SUPPORT_ONE) &&
+            getIntObjVar(attacker, ORIGINAL_CARBINE_SUPPORT_ONE) == 0 &&
+            hasSkill(attacker, CARBINE_SUPPORT_ONE))
+        {
+            revokeSkill(attacker, CARBINE_SUPPORT_ONE);
+        }
         if (hasObjVar(attacker, ORIGINAL_CARBINE_NOVICE) &&
             getIntObjVar(attacker, ORIGINAL_CARBINE_NOVICE) == 0 &&
             hasSkill(attacker, CARBINE_NOVICE))
@@ -2800,6 +2838,8 @@ public class precu_headshot1_fixture extends script.base_script
                 CARBINE_ACCURACY_TWO) &&
             isSkillRestored(attacker, ORIGINAL_CARBINE_ACCURACY_ONE,
                 CARBINE_ACCURACY_ONE) &&
+            isSkillRestored(attacker, ORIGINAL_CARBINE_SUPPORT_ONE,
+                CARBINE_SUPPORT_ONE) &&
             isSkillRestored(attacker, ORIGINAL_CARBINE_NOVICE,
                 CARBINE_NOVICE) &&
             isSkillRestored(attacker, ORIGINAL_CARBINE_FOUR,
@@ -2818,6 +2858,8 @@ public class precu_headshot1_fixture extends script.base_script
                 FULL_AUTO_SINGLE_ONE_COMMAND) &&
             isCommandRestored(attacker, ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND,
                 FULL_AUTO_SINGLE_TWO_COMMAND) &&
+            isCommandRestored(attacker, ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND,
+                FULL_AUTO_AREA_ONE_COMMAND) &&
             isCommandRestored(attacker, ORIGINAL_SCATTER_SHOT_ONE_COMMAND,
                 SCATTER_SHOT_ONE_COMMAND) &&
             isCommandRestored(attacker, ORIGINAL_SCATTER_SHOT_TWO_COMMAND,
@@ -2905,6 +2947,8 @@ public class precu_headshot1_fixture extends script.base_script
             CARBINE_ACCURACY_TWO);
         restoreSkill(attacker, ORIGINAL_CARBINE_ACCURACY_ONE,
             CARBINE_ACCURACY_ONE);
+        restoreSkill(attacker, ORIGINAL_CARBINE_SUPPORT_ONE,
+            CARBINE_SUPPORT_ONE);
         restoreSkill(attacker, ORIGINAL_CARBINE_NOVICE, CARBINE_NOVICE);
         restoreSkill(attacker, ORIGINAL_CARBINE_FOUR, MARKSMAN_CARBINE_FOUR);
         restoreSkill(attacker, ORIGINAL_CARBINE_THREE, MARKSMAN_CARBINE_THREE);
@@ -2916,6 +2960,8 @@ public class precu_headshot1_fixture extends script.base_script
             FULL_AUTO_SINGLE_ONE_COMMAND);
         restoreCommand(attacker, ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND,
             FULL_AUTO_SINGLE_TWO_COMMAND);
+        restoreCommand(attacker, ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND,
+            FULL_AUTO_AREA_ONE_COMMAND);
         restoreCommand(attacker, ORIGINAL_SCATTER_SHOT_ONE_COMMAND,
             SCATTER_SHOT_ONE_COMMAND);
         restoreCommand(attacker, ORIGINAL_SCATTER_SHOT_TWO_COMMAND,
@@ -3152,6 +3198,9 @@ public class precu_headshot1_fixture extends script.base_script
         boolean carbineAccuracyFourReady = carbineAccuracyThreeReady &&
             (hasSkill(attacker, CARBINE_ACCURACY_FOUR) ||
                 grantSkill(attacker, CARBINE_ACCURACY_FOUR));
+        boolean carbineSupportOneReady = carbineNoviceReady &&
+            (hasSkill(attacker, CARBINE_SUPPORT_ONE) ||
+                grantSkill(attacker, CARBINE_SUPPORT_ONE));
         boolean carbineSpeedOneReady = carbineNoviceReady &&
             (hasSkill(attacker, CARBINE_SPEED_ONE) ||
                 grantSkill(attacker, CARBINE_SPEED_ONE));
@@ -3271,6 +3320,9 @@ public class precu_headshot1_fixture extends script.base_script
         boolean fullAutoSingleTwoCommandReady =
             hasCommand(attacker, FULL_AUTO_SINGLE_TWO_COMMAND) ||
             grantCommand(attacker, FULL_AUTO_SINGLE_TWO_COMMAND);
+        boolean fullAutoAreaOneCommandReady =
+            hasCommand(attacker, FULL_AUTO_AREA_ONE_COMMAND) ||
+            grantCommand(attacker, FULL_AUTO_AREA_ONE_COMMAND);
         boolean scatterShotOneCommandReady =
             hasCommand(attacker, SCATTER_SHOT_ONE_COMMAND) ||
             grantCommand(attacker, SCATTER_SHOT_ONE_COMMAND);
@@ -3391,6 +3443,7 @@ public class precu_headshot1_fixture extends script.base_script
             carbineNoviceReady && carbineAccuracyOneReady &&
             carbineAccuracyTwoReady && carbineAccuracyThreeReady &&
             carbineAccuracyFourReady &&
+            carbineSupportOneReady &&
             carbineSpeedOneReady &&
             carbineAbilityFourReady &&
             supportOneReady && supportFourReady &&
@@ -3416,6 +3469,7 @@ public class precu_headshot1_fixture extends script.base_script
             kipUpShotCommandReady && takeCoverCommandReady &&
             fullAutoSingleOneCommandReady &&
             fullAutoSingleTwoCommandReady &&
+            fullAutoAreaOneCommandReady &&
             scatterShotOneCommandReady && scatterShotTwoCommandReady &&
             wildShotOneCommandReady && wildShotTwoCommandReady &&
             legShotTwoCommandReady && legShotThreeCommandReady &&
@@ -3705,6 +3759,7 @@ public class precu_headshot1_fixture extends script.base_script
             hasObjVar(player, ORIGINAL_CARBINE_ACCURACY_TWO) &&
             hasObjVar(player, ORIGINAL_CARBINE_ACCURACY_THREE) &&
             hasObjVar(player, ORIGINAL_CARBINE_ACCURACY_FOUR) &&
+            hasObjVar(player, ORIGINAL_CARBINE_SUPPORT_ONE) &&
             hasObjVar(player, ORIGINAL_CARBINE_SPEED_ONE) &&
             hasObjVar(player, ORIGINAL_CARBINE_ABILITY_ONE) &&
             hasObjVar(player, ORIGINAL_CARBINE_ABILITY_TWO) &&
@@ -3751,6 +3806,7 @@ public class precu_headshot1_fixture extends script.base_script
             hasObjVar(player, ORIGINAL_TAKE_COVER_COMMAND) &&
             hasObjVar(player, ORIGINAL_FULL_AUTO_SINGLE_ONE_COMMAND) &&
             hasObjVar(player, ORIGINAL_FULL_AUTO_SINGLE_TWO_COMMAND) &&
+            hasObjVar(player, ORIGINAL_FULL_AUTO_AREA_ONE_COMMAND) &&
             hasObjVar(player, ORIGINAL_SCATTER_SHOT_ONE_COMMAND) &&
             hasObjVar(player, ORIGINAL_SCATTER_SHOT_TWO_COMMAND) &&
             hasObjVar(player, ORIGINAL_WILD_SHOT_ONE_COMMAND) &&
@@ -3923,6 +3979,8 @@ public class precu_headshot1_fixture extends script.base_script
             combat.canPerformAction(FULL_AUTO_SINGLE_ONE_COMMAND, attacker) : -1;
         int canPerformFullAutoSingleTwo = isIdValid(weapon) ?
             combat.canPerformAction(FULL_AUTO_SINGLE_TWO_COMMAND, attacker) : -1;
+        int canPerformFullAutoAreaOne = isIdValid(weapon) ?
+            combat.canPerformAction(FULL_AUTO_AREA_ONE_COMMAND, attacker) : -1;
         int canPerformScatterShotOne = isIdValid(weapon) ?
             combat.canPerformAction(SCATTER_SHOT_ONE_COMMAND, attacker) : -1;
         int canPerformScatterShotTwo = isIdValid(weapon) ?
@@ -4042,6 +4100,8 @@ public class precu_headshot1_fixture extends script.base_script
             combat_engine.getCombatData(FULL_AUTO_SINGLE_ONE_COMMAND);
         combat_data fullAutoSingleTwoData =
             combat_engine.getCombatData(FULL_AUTO_SINGLE_TWO_COMMAND);
+        combat_data fullAutoAreaOneData =
+            combat_engine.getCombatData(FULL_AUTO_AREA_ONE_COMMAND);
         combat_data scatterShotOneData =
             combat_engine.getCombatData(SCATTER_SHOT_ONE_COMMAND);
         combat_data scatterShotTwoData =
@@ -4120,6 +4180,11 @@ public class precu_headshot1_fixture extends script.base_script
             combat.getActionCost(
                 attacker, flameWeaponData, fullAutoSingleTwoData) :
             new int[] {-1, -1, -1};
+        int[] fullAutoAreaOneCosts =
+            fullAutoAreaOneData != null && flameWeaponData != null ?
+            combat.getActionCost(
+                attacker, flameWeaponData, fullAutoAreaOneData) :
+            new int[] {-1, -1, -1};
         int[] scatterShotOneCosts =
             scatterShotOneData != null && flameWeaponData != null ?
             combat.getActionCost(
@@ -4182,6 +4247,8 @@ public class precu_headshot1_fixture extends script.base_script
                 hasSkill(attacker, CARBINE_ACCURACY_THREE) +
             " carbineAccuracyFour=" +
                 hasSkill(attacker, CARBINE_ACCURACY_FOUR) +
+            " carbineSupportOne=" +
+                hasSkill(attacker, CARBINE_SUPPORT_ONE) +
             " carbineSpeedOne=" + hasSkill(attacker, CARBINE_SPEED_ONE) +
             " carbineAbilityOne=" + hasSkill(attacker, CARBINE_ABILITY_ONE) +
             " carbineAbilityTwo=" + hasSkill(attacker, CARBINE_ABILITY_TWO) +
@@ -4250,6 +4317,8 @@ public class precu_headshot1_fixture extends script.base_script
                 hasCommand(attacker, FULL_AUTO_SINGLE_ONE_COMMAND) +
             " fullAutoSingleTwoCommand=" +
                 hasCommand(attacker, FULL_AUTO_SINGLE_TWO_COMMAND) +
+            " fullAutoAreaOneCommand=" +
+                hasCommand(attacker, FULL_AUTO_AREA_ONE_COMMAND) +
             " scatterShotOneCommand=" +
                 hasCommand(attacker, SCATTER_SHOT_ONE_COMMAND) +
             " scatterShotTwoCommand=" +
@@ -4359,6 +4428,7 @@ public class precu_headshot1_fixture extends script.base_script
             " canPerformTakeCover=" + canPerformTakeCover +
             " canPerformFullAutoSingleOne=" + canPerformFullAutoSingleOne +
             " canPerformFullAutoSingleTwo=" + canPerformFullAutoSingleTwo +
+            " canPerformFullAutoAreaOne=" + canPerformFullAutoAreaOne +
             " canPerformScatterShotOne=" + canPerformScatterShotOne +
             " canPerformScatterShotTwo=" + canPerformScatterShotTwo +
             " canPerformWildShotOne=" + canPerformWildShotOne +
@@ -4514,6 +4584,12 @@ public class precu_headshot1_fixture extends script.base_script
             " fullAutoSingleTwoPrecuHamCostModel=" +
                 (fullAutoSingleTwoData == null ? -1 :
                     fullAutoSingleTwoData.precuHamCostModel) +
+            " fullAutoAreaOneHealthCost=" + fullAutoAreaOneCosts[0] +
+            " fullAutoAreaOneActionCost=" + fullAutoAreaOneCosts[1] +
+            " fullAutoAreaOneMindCost=" + fullAutoAreaOneCosts[2] +
+            " fullAutoAreaOnePrecuHamCostModel=" +
+                (fullAutoAreaOneData == null ? -1 :
+                    fullAutoAreaOneData.precuHamCostModel) +
             " scatterShotOneHealthCost=" + scatterShotOneCosts[0] +
             " scatterShotOneActionCost=" + scatterShotOneCosts[1] +
             " scatterShotOneMindCost=" + scatterShotOneCosts[2] +
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
index b3027e4..086ba8f 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
@@ -1365,3 +1365,4 @@ fullAutoSingle2	fullAutoSingle2			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH
 suppressionFire2	suppressionFire2			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_defender_posture_change_down														OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	64	0	0	2.5	0.0	1.0	0	0	0	0	0	150	0.0	0	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				CARBINE	RANGED_WEAPON	ENERGY	KINETIC				marksman	0		ACTION_NAME	0	0	1
 wildShot1	wildShot1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_7_single						fire_7_single								OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	64	0	0	2.25	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				CARBINE	RANGED_WEAPON	ENERGY	KINETIC				carbineer	0		ACTION_NAME	0	0	1
 wildShot2	wildShot2			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_7_single						fire_7_single								OFF	OFF	OFF	OFF	OFF	CONE	64	30	0	64	0	0	3.0	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				CARBINE	RANGED_WEAPON	ENERGY	KINETIC				carbineer	0		ACTION_NAME	0	0	1
+fullAutoArea1	fullAutoArea1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_area							fire_area							OFF	OFF	OFF	OFF	OFF	CONE	64	30	0	64	0	0	1.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				CARBINE	RANGED_WEAPON	ENERGY	KINETIC				marksman	0		ACTION_NAME	0	0	1
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
index 377032e..0800b7f 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
@@ -81,3 +81,4 @@ fullAutoSingle2	2	2.5	.5	RANDOM	1.5	25	RANGED			DIZZY	30	0	30	dizzy_defense	jedi
 suppressionFire2	2	1.25	.5	RANDOM	1.5	25	NONE		100
 wildShot1	1.75	1.25	.5	RANDOM	2	25	RANGED			STUN	50	0	30	stun_defense	jedi_state_defense	resistance_states
 wildShot2	2.0	1.25	.5	RANDOM	2	25	RANGED			STUN	50	0	30	stun_defense	jedi_state_defense	resistance_states
+fullAutoArea1	1.75	2.5	0.5	RANDOM	1.5	25	INTENSITY			DIZZY	30	0	30	dizzy_defense	jedi_state_defense	resistance_states	BLIND	30	0	40	blind_defense	jedi_state_defense	resistance_states	STUN	30	0	30	stun_defense	jedi_state_defense	resistance_states
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
index 04ccf38..7f1c74e 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
@@ -80,3 +80,4 @@ fullAutoSingle2	s_auto
 suppressionFire2	sup_fire
 wildShot1	wildshot
 wildShot2	widewildshot
+fullAutoArea1	areashot
diff --git a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
index a914e16..76e6334 100644
--- a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
@@ -2015,3 +2015,4 @@ fullAutoSingle2	combat	normal	fullAutoSingle2	failSpecialAttack			1.5	fullAutoSi
 suppressionFire2	combat	normal	suppressionFire2	failSpecialAttack			1.5	suppressionFire2	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	1	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	0	0	0	0	0	0		other	optional		2	0	391413347	0	0	0	0	1	CARBINE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
 wildShot1	combat	normal	wildShot1	failSpecialAttack			1.5	wildShot1	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	1	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	0	0	0	0	0	0		other	optional		2	0	391413347	0	0	0	0	1	CARBINE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
 wildShot2	combat	normal	wildShot2	failSpecialAttack			1.5	wildShot2	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	1	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	0	0	0	0	0	0		other	optional		2	0	391413347	0	0	0	0	1	CARBINE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
+fullAutoArea1	combat	normal	fullAutoArea1	failSpecialAttack			0.0	fullAutoArea1	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	1	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	0	0	0	0	0	0		other	optional		2	0	391413347	0	0	0	0	1	CARBINE	NONE	defaultCooldownGroup	0	0.0	0		0	0	0
warning: in the working copy of 'dsrc/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'dsrc/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'dsrc/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'dsrc/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'dsrc/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'dsrc/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab', LF will be replaced by CRLF the next time Git touches it
