diff --git a/sku.0/sys.shared/compiled/game/datatables/buff/buff.tab b/sku.0/sys.shared/compiled/game/datatables/buff/buff.tab
--- a/sku.0/sys.shared/compiled/game/datatables/buff/buff.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/buff/buff.tab
@@ -1835,3 +1835,3 @@
 tcg_series3_hh_15_torpedo_warhead	gcw_exp_buff			10	buffs.tcg_series3_hh_15_torpedo_warhead	3600	tcg_gcw_bonus	10		0		0		0		0	STATE_NONE	none			1	0		0	0	1	0	1	1	0	1	1	1
-tcg_series3_wookiee_ceremonial_pipe	collection			20		1	random_collection_item	0		0		0		0		0	STATE_NONE	none			1	0		1	0	1	0	1	1	0	1	1	1
+tcg_series3_wookiee_ceremonial_pipe	collection			20	buffs.genericBuff	1	random_collection_item	0		0		0		0		0	STATE_NONE	none			1	0		1	0	1	0	1	1	0	1	1	1
 tcg_series3_hands_of_seduction	hands_effect			10	buffs.batSwarm	-1	looping_hands_particle	0		0		0		0		0	STATE_NONE	none	appearance/pt_tcg_hand_effect_01.prt,appearance/pt_tcg_hand_effect_01.prt	hold_l,hold_r	0	0		0	0	0	0	1	1	0	0	0	1
diff --git a/sku.0/sys.shared/compiled/game/datatables/buff/effect_mapping.tab b/sku.0/sys.shared/compiled/game/datatables/buff/effect_mapping.tab
--- a/sku.0/sys.shared/compiled/game/datatables/buff/effect_mapping.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/buff/effect_mapping.tab
@@ -704,0 +705,3 @@ private_center_of_being	skill	private_center_of_being	
+expertise_cooldown_line_sm_0_hope	skill	expertise_cooldown_line_sm_0_hope	
+lnd_dm_1	skill	lnd_dm_1	
+strength	attrib	strength	
