diff --git a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
--- a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
+++ b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
@@ -808,22 +808,6 @@
         }
         return SCRIPT_CONTINUE;
     }
-    public int sm_double_fire_recourse(obj_id self, dictionary params) throws InterruptedException
-    {
-        if (!buff.hasBuff(self, "sm_double_fire_recourse"))
-        {
-            buff.applyBuff(self, self, "sm_double_fire_recourse");
-        }
-        return SCRIPT_CONTINUE;
-    }
-    public int sm_false_hope_recourse(obj_id self, dictionary params) throws InterruptedException
-    {
-        if (!buff.hasBuff(self, "sm_false_hope_recourse"))
-        {
-            buff.applyBuff(self, self, "sm_false_hope_recourse");
-        }
-        return SCRIPT_CONTINUE;
-    }
     public int sm_break_the_deal_recourse(obj_id self, dictionary params) throws InterruptedException
     {
         if (!buff.hasBuff(self, "sm_break_the_deal_recourse"))
