diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
@@ -1,5 +1,7 @@
 actionName	combatSpam
 s	s
+creatureMeleeAttack	creature
+creatureRangedAttack	creature
 healthShot2	sapblast
 pistolMeleeDefense1	pistolwhip
 pistolMeleeDefense2	pistolwhip
@@ -28,6 +30,6 @@ polearmLegHit1	legbruiser
 polearmLunge1	lungestrike
 polearmLunge2	lungestrike
-unarmedLunge1	ridingbantha
+unarmedLunge1	lungeshiak
 unarmedLunge2	lungeshiak
 melee1hLunge1	lungestab
 melee1hLunge2	lungestab
