diff --git a/sku.0/sys.server/compiled/game/script/systems/combat/combat_player.java b/sku.0/sys.server/compiled/game/script/systems/combat/combat_player.java
--- a/sku.0/sys.server/compiled/game/script/systems/combat/combat_player.java
+++ b/sku.0/sys.server/compiled/game/script/systems/combat/combat_player.java
@@ -7,5 +7,11 @@ public class combat_player extends script.systems.combat.combat_base
 {
     public combat_player()
     {
     }
+    public int peace(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
+    {
+        queueClear(self);
+        stopCombat(self);
+        return SCRIPT_CONTINUE;
+    }
     public int OnExitedCombat(obj_id self) throws InterruptedException
diff --git a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
index 149881a01..08820e300 100755
--- a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
@@ -1044,6 +1044,7 @@ tellpet		immediate	cmdTellPet				1																								0	1						0						1				0
 tip		immediate	cmdTip																												0	1												1																														optional		2			0				0										
 toggleAwayFromKeyBoard		immediate			toggleAwayFromKeyBoard																							1	1		0	1												1																														none					0				0										
 toggleCombatTaunts		immediate	toggleCombatTaunts				0																								0	1												1																														optional					0				0										
+peace		immediate	peace				0																								0	1												1																														none					0				0									0	0
 toggleDisplayingFactionRank		immediate			toggleDisplayingFactionRank																							1	1		0	1												1																														none					0				0										
 trainPet		normal	cmdTrainPet				0.5																		0	0	0				0	1				0		0						1	0	0	0	0	0			0							0		0	0	0	0										optional		2							0									1	
 transferItem		immediate			transferItem																										0	1												1																						0								required					0				0										
