diff --git a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
index 5ad3354bf..ec66e70ca 100755
--- a/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/skill/skills.tab
@@ -714,16 +714,16 @@ social_language_hutt	social_language		0	0	0	0		0					0	0										0	0
 social_language_hutt_speak	social_language		0	0	0	0	0	0					0	0				"bothan,human,moncalamari,rodian,trandoshan,twilek,zabrak"			language+huttese	language_hutt_speak=100			0	0
 social_language_hutt_comprehend	social_language		0	0	0	0	0	0					0	0								language_hutt_comprehend=100			0	0
 species			1	0	0	0	0	0						0											0	0
-species_bothan	species		1	0	0	0	0	0				unobtainable	1	0				bothan			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_bothan	species		1	0	0	0	0	0				unobtainable	1	0				bothan			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_human	species		1	0	0	0	0	0				unobtainable	1	0				human			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_human	species		1	0	0	0	0	0				unobtainable	1	0				human			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_moncal	species		1	0	0	0	0	0				unobtainable	1	0				moncal			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_moncal	species		1	0	0	0	0	0				unobtainable	1	0				moncal			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_rodian	species		1	0	0	0	0	0				unobtainable	1	0				rodian			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_rodian	species		1	0	0	0	0	0				unobtainable	1	0				rodian			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_trandoshan	species		1	0	0	0	0	0				unobtainable	1	0				trandoshan			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_trandoshan	species		1	0	0	0	0	0				unobtainable	1	0				trandoshan			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_twilek	species		1	0	0	0	0	0				unobtainable	1	0				twilek			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_twilek	species		1	0	0	0	0	0				unobtainable	1	0				twilek			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_wookiee	species		1	0	0	0	0	0				unobtainable	1	0				wookiee			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	"craftWookieItems,craftArtisanFreebieGroupA"		0	0
+species_wookiee	species		1	0	0	0	0	0				unobtainable	1	0				wookiee			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		"craftWookieItems,craftArtisanFreebieGroupA"		0	0
-species_zabrak	species		1	0	0	0	0	0				unobtainable	1	0				zabrak			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_zabrak	species		1	0	0	0	0	0				unobtainable	1	0				zabrak			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_ithorian	species		1	0	0	0	0	0				unobtainable	1	0				ithorian			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_ithorian	species		1	0	0	0	0	0				unobtainable	1	0				ithorian			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
-species_sullustan	species		1	0	0	0	0	0				unobtainable	1	0				sullustan			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"	creature_harvesting=25	craftArtisanFreebieGroupA		0	0
+species_sullustan	species		1	0	0	0	0	0				unobtainable	1	0				sullustan			"startDance,stopDance,startMusic,stopMustic,startDance+basic,startMusic+starwars1,slitherhorn,cert_ordnance_level1"		craftArtisanFreebieGroupA		0	0
 DEPRECATED_SKILLS			1	0	0	1	0	0						0											0	0
 outdoors			1	0	0	1	0	0						0											0	0
 outdoors_scout	outdoors	fourByFour	0	0	1	0	0	0	0				0	0		0			none						1	0
diff --git a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
index da61d2ff9..a71b7b211 100755
--- a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
@@ -778,2 +778,2 @@ harmless		immediate	cmdHarmless	failAdmin				admin																				1	1		0	1
-harvestCorpse		normal	cmdHarvestCorpse	cmdHarvestCorpseFail			1		1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	1	1	0			0	0	0	0	1	0	0	1	0	1	1	1	0	0	1	0	1	1	0	0	0									1		0	0	0	0								player.skill.outdoorsman		required				skill_outdoorsman	0	16			1										
+harvestCorpse		normal	cmdHarvestCorpse	cmdHarvestCorpseFail			1	harvestCorpse	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	1	1	0			0	0	0	0	1	0	0	1	0	1	1	1	0	0	1	0	1	1	0	0	0									1		0	0	0	0								player.skill.outdoorsman		required				skill_outdoorsman	0	16			1										
 harvesterActivate		immediate			harvesterActivate		0																					1	1		0	1					0							1																														optional				resource	0				0										
