diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_armorsmith_droid.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_armorsmith_droid.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_armorsmith_droid.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_armorsmith_droid.java
@@ -36,7 +36,7 @@ public class craft_armorsmith_droid extends script.base_script
         {
             return SCRIPT_CONTINUE;
         }
-        if (!hasSkill(giver, "class_munitions_phase4_master"))
+        if (!hasSkill(giver, "crafting_armorsmith_master"))
         {
             CustomerServiceLog("DUNGEON_DeathWatchBunker", "*Mandalorian Armor: Player %TU was not a crafting armorsmith master, so erroring out.");
             string_id master_armorsmith = new string_id(MSGS, "master_armorsmith_required");
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_droidengineer_droid.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_droidengineer_droid.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_droidengineer_droid.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_droidengineer_droid.java
@@ -36,7 +36,7 @@ public class craft_droidengineer_droid extends script.base_script
         {
             return SCRIPT_CONTINUE;
         }
-        if (!hasSkill(giver, "class_engineering_phase4_master"))
+        if (!hasSkill(giver, "crafting_droidengineer_master"))
         {
             CustomerServiceLog("DUNGEON_DeathWatchBunker", "*Mandalorian Armor: Player %TU was not a crafting droidengineer master, so erroring out.");
             string_id master_droidengineer = new string_id(MSGS, "master_droidengineer_required");
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_jetpack_droid.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_jetpack_droid.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_jetpack_droid.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_jetpack_droid.java
@@ -37,7 +37,7 @@ public class craft_jetpack_droid extends script.base_script
         {
             return SCRIPT_CONTINUE;
         }
-        if (!hasSkill(giver, "class_engineering_phase1_master"))
+        if (!hasSkill(giver, "crafting_artisan_master"))
         {
             CustomerServiceLog("DUNGEON_DeathWatchBunker", "*Mandalorian Armor: Player %TU was not a crafting artisan master, so erroring out.");
             string_id master_tailor = new string_id(MSGS, "master_artisan_required");
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_tailor_droid.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_tailor_droid.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_tailor_droid.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/craft_tailor_droid.java
@@ -36,7 +36,7 @@ public class craft_tailor_droid extends script.base_script
         {
             return SCRIPT_CONTINUE;
         }
-        if (!hasSkill(giver, "class_domestics_phase4_master"))
+        if (!hasSkill(giver, "crafting_tailor_master"))
         {
             CustomerServiceLog("DUNGEON_DeathWatchBunker", "*Mandalorian Armor: Player %TU was not a crafting tailor master, so erroring out.");
             string_id master_tailor = new string_id(MSGS, "master_tailor_required");
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_armor.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_armor.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_armor.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_armor.java
@@ -30,7 +30,7 @@ public class door_lock_crafting_armor extends script.base_script
                 messageTo(self, "unlockTerminal", null, 30, false);
                 return SCRIPT_CONTINUE;
             }
-            if (hasSkill(player, "class_munitions_phase4_master"))
+            if (hasSkill(player, "crafting_armorsmith_master"))
             {
                 if (checkForIngredients(player))
                 {
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_de.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_de.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_de.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_de.java
@@ -30,7 +30,7 @@ public class door_lock_crafting_de extends script.base_script
                 messageTo(self, "unlockTerminal", null, 30, false);
                 return SCRIPT_CONTINUE;
             }
-            if (hasSkill(player, "class_engineering_phase4_master"))
+            if (hasSkill(player, "crafting_droidengineer_master"))
             {
                 if (checkForIngredients(player))
                 {
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_tailor.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_tailor.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_tailor.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/death_watch_bunker/door_lock_crafting_tailor.java
@@ -30,7 +30,7 @@ public class door_lock_crafting_tailor extends script.base_script
                 messageTo(self, "unlockTerminal", null, 30, false);
                 return SCRIPT_CONTINUE;
             }
-            if (hasSkill(player, "class_domestics_phase4_master"))
+            if (hasSkill(player, "crafting_tailor_master"))
             {
                 if (checkForIngredients(player))
                 {
diff --git a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/mustafar_trials/valley_battleground/mining_droid.java b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/mustafar_trials/valley_battleground/mining_droid.java
--- a/sku.0/sys.server/compiled/game/script/theme_park/dungeon/mustafar_trials/valley_battleground/mining_droid.java
+++ b/sku.0/sys.server/compiled/game/script/theme_park/dungeon/mustafar_trials/valley_battleground/mining_droid.java
@@ -79,7 +79,7 @@ public class mining_droid extends script.base_script
     }
     public boolean isDroidEngineer(obj_id player) throws InterruptedException
     {
-        return hasSkill(player, "class_engineering_phase2_novice");
+        return hasSkill(player, "crafting_droidengineer_novice");
     }
     public void activateDroid(obj_id self) throws InterruptedException
     {
diff --git a/sku.0/sys.server/compiled/game/script/npc/static_quest/quest_armorsmith.java b/sku.0/sys.server/compiled/game/script/npc/static_quest/quest_armorsmith.java
--- a/sku.0/sys.server/compiled/game/script/npc/static_quest/quest_armorsmith.java
+++ b/sku.0/sys.server/compiled/game/script/npc/static_quest/quest_armorsmith.java
@@ -441,7 +441,7 @@ public class quest_armorsmith extends script.base_script
         {
             return -1;
         }
-        if (!hasSkill(player, "class_munitions_phase4_master"))
+        if (!hasSkill(player, "crafting_armorsmith_master"))
         {
             return -1;
         }
