diff --git a/sku.0/sys.server/compiled/game/script/npc/pet_deed/droid_deed.java b/sku.0/sys.server/compiled/game/script/npc/pet_deed/droid_deed.java
index 1fe0f94..bbc71f6 100644
--- a/sku.0/sys.server/compiled/game/script/npc/pet_deed/droid_deed.java
+++ b/sku.0/sys.server/compiled/game/script/npc/pet_deed/droid_deed.java
@@ -710,12 +710,6 @@ public class droid_deed extends script.base_script
         {
             String creatureName = getStringObjVar(self, "creature_attribs.type");
             int petType = pet_lib.getPetType(creatureName);
-            int droidLevel = getIntObjVar(self, "creature_attribs.level");
-            if (getLevel(player) < droidLevel - pet_lib.MAX_PET_LEVELS_ABOVE_CALLER)
-            {
-                sendSystemMessage(player, pet_lib.SID_SYS_CANT_CALL_LEVEL);
-                return SCRIPT_CONTINUE;
-            }
             if (pet_lib.hasMaxStoredPetsOfType(player, petType))
             {
                 sendSystemMessage(player, beast_lib.SID_MAXIMUM_COMBAT_CONTROL_DEVICES);
