diff --git a/sku.0/sys.server/compiled/game/script/player/base/base_player.java b/sku.0/sys.server/compiled/game/script/player/base/base_player.java
--- a/sku.0/sys.server/compiled/game/script/player/base/base_player.java
+++ b/sku.0/sys.server/compiled/game/script/player/base/base_player.java
@@ -11661,8 +11661,9 @@
         }
         if (regionName.equals(restuss_event.PVP_REGION_NAME))
         {
-            if ((!factions.isImperial(self) && !factions.isRebel(self)) || !factions.isCovert(self) || getLevel(self) < 75)
+            int precuCombatDifficulty = skill.getPrecuEncounterDifficulty(self);
+            if ((!factions.isImperial(self) && !factions.isRebel(self)) || !factions.isCovert(self) || precuCombatDifficulty < 75)
             {
-                if (getLevel(self) < 75)
+                if (precuCombatDifficulty < 75)
                 {
                     sendSystemMessage(self, new string_id("gcw", "pvp_advanced_region_level_low"));
