diff --git a/game/server/database/build/linux/getversion.sql b/game/server/database/build/linux/getversion.sql
--- a/game/server/database/build/linux/getversion.sql
+++ b/game/server/database/build/linux/getversion.sql
@@ -1 +1,7 @@
-select version_number from version_number;
\ No newline at end of file
+set heading on
+set feedback off
+set pagesize 100
+
+select version_number from version_number;
+
+exit;
