diff --git a/engine/server/library/serverGame/src/shared/object/GroupObject.cpp b/engine/server/library/serverGame/src/shared/object/GroupObject.cpp
index 93ee461d72181d717980064c397972f56a124982..df265f846efbb8966a92bfa89c5a3f9d0af08b46 100755
--- a/engine/server/library/serverGame/src/shared/object/GroupObject.cpp
+++ b/engine/server/library/serverGame/src/shared/object/GroupObject.cpp
@@ -55,7 +55,7 @@ namespace GroupObjectNamespace
 {
 	// ----------------------------------------------------------------------
 
-	const uint32_t cs_maximumNumberInGroup = 8;
+	const uint32_t cs_maximumNumberInGroup = 24;
 	char const * const DEFAULT_GROUP_TEMPLATE = "object/group/group.iff";
 	std::map<NetworkId, NetworkId> s_leaderMap;
 	static const std::string cs_emptyString;
